<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 GM - Site/Raw Material | Caddo retouch results in divergent morphology for elliptical bifaces from sites in the American Southeast</title>
  <meta name="description" content="Pilot study of biface morphology." />
  <meta name="generator" content="bookdown 0.27 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 GM - Site/Raw Material | Caddo retouch results in divergent morphology for elliptical bifaces from sites in the American Southeast" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Pilot study of biface morphology." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 GM - Site/Raw Material | Caddo retouch results in divergent morphology for elliptical bifaces from sites in the American Southeast" />
  
  <meta name="twitter:description" content="Pilot study of biface morphology." />
  

<meta name="author" content="Robert Z. Selden, Jr. and John E. Dockall" />


<meta name="date" content="2022-08-05" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="define-size-classes.html"/>
<link rel="next" href="gm---size-class.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="custom.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">jowell.1</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a>
<ul>
<li class="chapter" data-level="0.1" data-path="index.html"><a href="index.html#analysis-overview"><i class="fa fa-check"></i><b>0.1</b> Analysis overview</a>
<ul>
<li class="chapter" data-level="0.1.1" data-path="index.html"><a href="index.html#generalised-procrustes-analysis"><i class="fa fa-check"></i><b>0.1.1</b> Generalised Procrustes Analysis</a></li>
<li class="chapter" data-level="0.1.2" data-path="index.html"><a href="index.html#principal-components-analysis"><i class="fa fa-check"></i><b>0.1.2</b> Principal Components Analysis</a></li>
<li class="chapter" data-level="0.1.3" data-path="index.html"><a href="index.html#procrustes-anova"><i class="fa fa-check"></i><b>0.1.3</b> Procrustes ANOVA</a></li>
<li class="chapter" data-level="0.1.4" data-path="index.html"><a href="index.html#morhpological-disparity"><i class="fa fa-check"></i><b>0.1.4</b> Morhpological disparity</a></li>
</ul></li>
<li class="chapter" data-level="0.2" data-path="index.html"><a href="index.html#primary-findings"><i class="fa fa-check"></i><b>0.2</b> Primary findings</a>
<ul>
<li class="chapter" data-level="0.2.1" data-path="index.html"><a href="index.html#summary"><i class="fa fa-check"></i><b>0.2.1</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="0.3" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i><b>0.3</b> Acknowledgments</a></li>
<li class="chapter" data-level="0.4" data-path="index.html"><a href="index.html#funding"><i class="fa fa-check"></i><b>0.4</b> Funding</a></li>
<li class="chapter" data-level="0.5" data-path="index.html"><a href="index.html#data-management"><i class="fa fa-check"></i><b>0.5</b> Data management</a></li>
<li class="chapter" data-level="0.6" data-path="index.html"><a href="index.html#colophon"><i class="fa fa-check"></i><b>0.6</b> Colophon</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="define-size-classes.html"><a href="define-size-classes.html"><i class="fa fa-check"></i><b>1</b> Define size classes</a>
<ul>
<li class="chapter" data-level="1.1" data-path="define-size-classes.html"><a href="define-size-classes.html#load-packages-data"><i class="fa fa-check"></i><b>1.1</b> Load packages + data</a></li>
<li class="chapter" data-level="1.2" data-path="define-size-classes.html"><a href="define-size-classes.html#generalised-procrustes-analysis-1"><i class="fa fa-check"></i><b>1.2</b> Generalised Procrustes Analysis</a></li>
<li class="chapter" data-level="1.3" data-path="define-size-classes.html"><a href="define-size-classes.html#assign-size-class"><i class="fa fa-check"></i><b>1.3</b> Assign size class</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html"><i class="fa fa-check"></i><b>2</b> GM - Site/Raw Material</a>
<ul>
<li class="chapter" data-level="2.1" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html#load-packages-data-1"><i class="fa fa-check"></i><b>2.1</b> Load packages + data</a></li>
<li class="chapter" data-level="2.2" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html#generalised-procrustes-analysis-2"><i class="fa fa-check"></i><b>2.2</b> Generalised Procrustes Analysis</a></li>
<li class="chapter" data-level="2.3" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html#principal-components-analysis-1"><i class="fa fa-check"></i><b>2.3</b> Principal Components Analysis</a></li>
<li class="chapter" data-level="2.4" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html#procrustes-anova-1"><i class="fa fa-check"></i><b>2.4</b> Procrustes ANOVA</a></li>
<li class="chapter" data-level="2.5" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html#morphological-disparity"><i class="fa fa-check"></i><b>2.5</b> Morphological Disparity</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="gm---size-class.html"><a href="gm---size-class.html"><i class="fa fa-check"></i><b>3</b> GM - Size Class</a>
<ul>
<li class="chapter" data-level="3.1" data-path="gm---size-class.html"><a href="gm---size-class.html#load-packages-data-2"><i class="fa fa-check"></i><b>3.1</b> Load packages + data</a></li>
<li class="chapter" data-level="3.2" data-path="gm---size-class.html"><a href="gm---size-class.html#generalised-procrustes-analysis-3"><i class="fa fa-check"></i><b>3.2</b> Generalised Procrustes Analysis</a></li>
<li class="chapter" data-level="3.3" data-path="gm---size-class.html"><a href="gm---size-class.html#principal-components-analysis-2"><i class="fa fa-check"></i><b>3.3</b> Principal Components Analysis</a></li>
<li class="chapter" data-level="3.4" data-path="gm---size-class.html"><a href="gm---size-class.html#procrustes-anova-2"><i class="fa fa-check"></i><b>3.4</b> Procrustes ANOVA</a></li>
<li class="chapter" data-level="3.5" data-path="gm---size-class.html"><a href="gm---size-class.html#morphological-disparity-1"><i class="fa fa-check"></i><b>3.5</b> Morphological Disparity</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Caddo retouch results in divergent morphology for elliptical bifaces from sites in the American Southeast</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="gm---siteraw-material" class="section level1 hasAnchor" number="2">
<h1><span class="header-section-number">Chapter 2</span> GM - Site/Raw Material<a href="gm---siteraw-material.html#gm---siteraw-material" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Elliptical bifaces from the Millsap Cache and Jowell Farm were used to examine whether their morphology remains stable, or expresses morphological variability by site/raw meterial. Variability in elliptical biface morphology provides evidence associated with differences in morphology as a function of geography/raw material.</p>
<p>This analysis follows recent analyses of Perdiz arrow points (<a href="https://github.com/seldenlab/perdiz3/blob/main/ms/perdiz3/perdiz3.pdf">Selden and Dockall, in review</a>) and Gahagan biface morphology across the southern Caddo area and central Texas <span class="citation">(<a href="#ref-RN11783" role="doc-biblioref">Selden Jr., Dockall, and Shafer 2018</a>; <a href="#ref-RN21001" role="doc-biblioref">Selden Jr., Dockall, and Dubied 2020</a>; <a href="#ref-RN11264" role="doc-biblioref">Selden 2022</a>)</span>, where Perdiz arrow points and Gahagan bifaces have been found to differ in shape across the same geography as Caddo bottles <span class="citation">(<a href="#ref-RN11801" role="doc-biblioref">Selden Jr. 2018a</a>, <a href="#ref-RN11782" role="doc-biblioref">2018b</a>, <a href="#ref-RN11716" role="doc-biblioref">2019</a>, <a href="#ref-RN8312" role="doc-biblioref">2021</a>)</span>. This analysis also follows another geometric morphometric analysis in which Perdiz arrow points from the ancestral Caddo area were found to differ as a function of <em>time</em>, <em>raw material</em>, and <em>burial context</em> <span class="citation">(<a href="#ref-RN9551" role="doc-biblioref">Selden Jr et al. 2021b</a>)</span>.</p>
<div id="load-packages-data-1" class="section level2 hasAnchor" number="2.1">
<h2><span class="header-section-number">2.1</span> Load packages + data<a href="gm---siteraw-material.html#load-packages-data-1" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="gm---siteraw-material.html#cb12-1" aria-hidden="true" tabindex="-1"></a><span class="co"># load required analysis packages</span></span>
<span id="cb12-2"><a href="gm---siteraw-material.html#cb12-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(here)</span>
<span id="cb12-3"><a href="gm---siteraw-material.html#cb12-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(StereoMorph)</span>
<span id="cb12-4"><a href="gm---siteraw-material.html#cb12-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(geomorph)</span>
<span id="cb12-5"><a href="gm---siteraw-material.html#cb12-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb12-6"><a href="gm---siteraw-material.html#cb12-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(wesanderson)</span>
<span id="cb12-7"><a href="gm---siteraw-material.html#cb12-7" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb12-8"><a href="gm---siteraw-material.html#cb12-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-9"><a href="gm---siteraw-material.html#cb12-9" aria-hidden="true" tabindex="-1"></a><span class="co"># read data and define number of sLMs ----</span></span>
<span id="cb12-10"><a href="gm---siteraw-material.html#cb12-10" aria-hidden="true" tabindex="-1"></a>shapes <span class="ot">&lt;-</span> <span class="fu">readShapes</span>(<span class="st">&quot;shapes&quot;</span>)</span>
<span id="cb12-11"><a href="gm---siteraw-material.html#cb12-11" aria-hidden="true" tabindex="-1"></a>shapesGM <span class="ot">&lt;-</span> <span class="fu">readland.shapes</span>(shapes, <span class="at">nCurvePts =</span> <span class="fu">c</span>(<span class="dv">10</span>, <span class="dv">10</span>, <span class="dv">15</span>))</span>
<span id="cb12-12"><a href="gm---siteraw-material.html#cb12-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-13"><a href="gm---siteraw-material.html#cb12-13" aria-hidden="true" tabindex="-1"></a><span class="co"># read qualitative data</span></span>
<span id="cb12-14"><a href="gm---siteraw-material.html#cb12-14" aria-hidden="true" tabindex="-1"></a>qdata <span class="ot">&lt;-</span> <span class="fu">read.csv</span>(<span class="st">&quot;qdata.merged.csv&quot;</span>, </span>
<span id="cb12-15"><a href="gm---siteraw-material.html#cb12-15" aria-hidden="true" tabindex="-1"></a>                  <span class="at">header =</span> <span class="cn">TRUE</span>, </span>
<span id="cb12-16"><a href="gm---siteraw-material.html#cb12-16" aria-hidden="true" tabindex="-1"></a>                  <span class="at">row.names =</span> <span class="dv">1</span>)</span></code></pre></div>
</div>
<div id="generalised-procrustes-analysis-2" class="section level2 hasAnchor" number="2.2">
<h2><span class="header-section-number">2.2</span> Generalised Procrustes Analysis<a href="gm---siteraw-material.html#generalised-procrustes-analysis-2" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Landmark data were aligned to a global coordinate system <span class="citation">(<a href="#ref-RN11622" role="doc-biblioref">Kendall 1981</a>, <a href="#ref-RN11623" role="doc-biblioref">1984</a>; <a href="#ref-RN11563" role="doc-biblioref">Slice 2001</a>)</span>, achieved through generalized Procrustes superimposition <span class="citation">(<a href="#ref-RN478" role="doc-biblioref">Rohlf and Slice 1990</a>)</span> performed in R 4.1.3 <span class="citation">(<a href="#ref-R" role="doc-biblioref">R Core Development Team, 2022</a>)</span> using the <code>geomorph</code> library v. 4.0.3 <span class="citation">(<a href="#ref-RN11530" role="doc-biblioref">Adams et al. 2017</a>; <a href="#ref-RN1774" role="doc-biblioref">Adams and Otarola-Castillo 2013</a>; <a href="#ref-RN9565" role="doc-biblioref">Baken et al. 2021</a>)</span>. Procrustes superimposition translates, scales, and rotates the coordinate data to allow for comparisons among objects <span class="citation">(<a href="#ref-RN11564" role="doc-biblioref">Gower 1975</a>; <a href="#ref-RN478" role="doc-biblioref">Rohlf and Slice 1990</a>)</span>. The <code>geomorph</code> package uses a partial Procrustes superimposition that projects the aligned specimens into tangent space subsequent to alignment in preparation for the use of multivariate methods that assume linear space <span class="citation">(<a href="#ref-RN1646" role="doc-biblioref">Rohlf 1999</a>; <a href="#ref-RN11563" role="doc-biblioref">Slice 2001</a>)</span>.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="gm---siteraw-material.html#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="co"># gpa ----</span></span>
<span id="cb13-2"><a href="gm---siteraw-material.html#cb13-2" aria-hidden="true" tabindex="-1"></a>Y.gpa <span class="ot">&lt;-</span> <span class="fu">gpagen</span>(shapesGM, <span class="at">print.progress =</span> <span class="cn">FALSE</span>)</span>
<span id="cb13-3"><a href="gm---siteraw-material.html#cb13-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-4"><a href="gm---siteraw-material.html#cb13-4" aria-hidden="true" tabindex="-1"></a><span class="co"># plot</span></span>
<span id="cb13-5"><a href="gm---siteraw-material.html#cb13-5" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(Y.gpa)</span></code></pre></div>
<p><img src="supp_files/figure-html/outlinessiteraw-1.png" width="100%" /></p>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="gm---siteraw-material.html#cb14-1" aria-hidden="true" tabindex="-1"></a><span class="co"># geomorph data frame ----</span></span>
<span id="cb14-2"><a href="gm---siteraw-material.html#cb14-2" aria-hidden="true" tabindex="-1"></a>gdf <span class="ot">&lt;-</span> <span class="fu">geomorph.data.frame</span>(<span class="at">shape =</span> Y.gpa<span class="sc">$</span>coords, </span>
<span id="cb14-3"><a href="gm---siteraw-material.html#cb14-3" aria-hidden="true" tabindex="-1"></a>                           <span class="at">size =</span> Y.gpa<span class="sc">$</span>Csize,</span>
<span id="cb14-4"><a href="gm---siteraw-material.html#cb14-4" aria-hidden="true" tabindex="-1"></a>                           <span class="at">site =</span> qdata<span class="sc">$</span>site)</span>
<span id="cb14-5"><a href="gm---siteraw-material.html#cb14-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-6"><a href="gm---siteraw-material.html#cb14-6" aria-hidden="true" tabindex="-1"></a><span class="co"># add centroid size to qdata</span></span>
<span id="cb14-7"><a href="gm---siteraw-material.html#cb14-7" aria-hidden="true" tabindex="-1"></a>qdata<span class="sc">$</span>csz <span class="ot">&lt;-</span> Y.gpa<span class="sc">$</span>Csize</span></code></pre></div>
</div>
<div id="principal-components-analysis-1" class="section level2 hasAnchor" number="2.3">
<h2><span class="header-section-number">2.3</span> Principal Components Analysis<a href="gm---siteraw-material.html#principal-components-analysis-1" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Principal components analysis <span class="citation">(<a href="#ref-RN1746" role="doc-biblioref">Jolliffe 2002</a>)</span> was used to visualise shape variation among the Perdiz arrow points. The shape changes described by each principal axis are commonly visualized using thin-plate spline warping of a reference 3D mesh <span class="citation">(<a href="#ref-RN1731" role="doc-biblioref">Klingenberg 2013</a>; <a href="#ref-RN479" role="doc-biblioref">Sherratt et al. 2014</a>)</span>.</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="gm---siteraw-material.html#cb15-1" aria-hidden="true" tabindex="-1"></a><span class="co"># principal components analysis ----</span></span>
<span id="cb15-2"><a href="gm---siteraw-material.html#cb15-2" aria-hidden="true" tabindex="-1"></a>pca<span class="ot">&lt;-</span><span class="fu">gm.prcomp</span>(Y.gpa<span class="sc">$</span>coords)</span>
<span id="cb15-3"><a href="gm---siteraw-material.html#cb15-3" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(pca)</span></code></pre></div>
<pre><code>## 
## Ordination type: Principal Component Analysis 
## Centering by OLS mean
## Orthogonal projection of OLS residuals
## Number of observations: 30 
## Number of vectors 29 
## 
## Importance of Components:
##                              Comp1        Comp2        Comp3        Comp4        Comp5        Comp6
## Eigenvalues            0.006537853 0.0003781135 0.0003022611 0.0000559455 2.637868e-05 1.078776e-05
## Proportion of Variance 0.890791573 0.0515184909 0.0411834968 0.0076226514 3.594131e-03 1.469847e-03
## Cumulative Proportion  0.890791573 0.9423100637 0.9834935605 0.9911162119 9.947103e-01 9.961802e-01
##                               Comp7        Comp8        Comp9       Comp10       Comp11       Comp12
## Eigenvalues            1.009261e-05 4.982089e-06 4.101647e-06 2.308128e-06 2.128514e-06 1.363224e-06
## Proportion of Variance 1.375132e-03 6.788165e-04 5.588550e-04 3.144856e-04 2.900130e-04 1.857412e-04
## Cumulative Proportion  9.975553e-01 9.982341e-01 9.987930e-01 9.991075e-01 9.993975e-01 9.995832e-01
##                              Comp13       Comp14       Comp15       Comp16       Comp17       Comp18
## Eigenvalues            9.015868e-07 8.306757e-07 3.910904e-07 2.564709e-07 2.022720e-07 1.312004e-07
## Proportion of Variance 1.228424e-04 1.131807e-04 5.328661e-05 3.494452e-05 2.755984e-05 1.787624e-05
## Cumulative Proportion  9.997061e-01 9.998193e-01 9.998725e-01 9.999075e-01 9.999350e-01 9.999529e-01
##                              Comp19       Comp20       Comp21       Comp22       Comp23       Comp24
## Eigenvalues            1.084451e-07 5.534632e-08 5.051539e-08 4.411557e-08 3.291932e-08 2.804381e-08
## Proportion of Variance 1.477579e-05 7.541013e-06 6.882792e-06 6.010807e-06 4.485303e-06 3.821008e-06
## Cumulative Proportion  9.999677e-01 9.999752e-01 9.999821e-01 9.999881e-01 9.999926e-01 9.999964e-01
##                              Comp25       Comp26       Comp27       Comp28       Comp29
## Eigenvalues            1.071824e-08 7.727764e-09 3.729096e-09 2.821059e-09 1.127017e-09
## Proportion of Variance 1.460375e-06 1.052919e-06 5.080945e-07 3.843732e-07 1.535576e-07
## Cumulative Proportion  9.999979e-01 9.999990e-01 9.999995e-01 9.999998e-01 1.000000e+00</code></pre>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="gm---siteraw-material.html#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="co"># set plot parameters</span></span>
<span id="cb17-2"><a href="gm---siteraw-material.html#cb17-2" aria-hidden="true" tabindex="-1"></a>pch.gps <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">2</span>)[<span class="fu">as.factor</span>(qdata<span class="sc">$</span>site)]</span>
<span id="cb17-3"><a href="gm---siteraw-material.html#cb17-3" aria-hidden="true" tabindex="-1"></a>col.gps <span class="ot">&lt;-</span> <span class="fu">wes_palette</span>(<span class="st">&quot;Moonrise2&quot;</span>)[<span class="fu">as.factor</span>(qdata<span class="sc">$</span>site)]</span>
<span id="cb17-4"><a href="gm---siteraw-material.html#cb17-4" aria-hidden="true" tabindex="-1"></a>col.hull <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">&quot;#C27D38&quot;</span>, <span class="st">&quot;#798E87&quot;</span>)</span>
<span id="cb17-5"><a href="gm---siteraw-material.html#cb17-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-6"><a href="gm---siteraw-material.html#cb17-6" aria-hidden="true" tabindex="-1"></a><span class="co"># plot pca by merged</span></span>
<span id="cb17-7"><a href="gm---siteraw-material.html#cb17-7" aria-hidden="true" tabindex="-1"></a>pc.plot <span class="ot">&lt;-</span> <span class="fu">plot</span>(pca, </span>
<span id="cb17-8"><a href="gm---siteraw-material.html#cb17-8" aria-hidden="true" tabindex="-1"></a>                <span class="at">asp =</span> <span class="dv">1</span>,</span>
<span id="cb17-9"><a href="gm---siteraw-material.html#cb17-9" aria-hidden="true" tabindex="-1"></a>                <span class="at">pch =</span> pch.gps,</span>
<span id="cb17-10"><a href="gm---siteraw-material.html#cb17-10" aria-hidden="true" tabindex="-1"></a>                <span class="at">col =</span> col.gps)</span>
<span id="cb17-11"><a href="gm---siteraw-material.html#cb17-11" aria-hidden="true" tabindex="-1"></a><span class="fu">shapeHulls</span>(pc.plot, </span>
<span id="cb17-12"><a href="gm---siteraw-material.html#cb17-12" aria-hidden="true" tabindex="-1"></a>           <span class="at">groups =</span> qdata<span class="sc">$</span>site,</span>
<span id="cb17-13"><a href="gm---siteraw-material.html#cb17-13" aria-hidden="true" tabindex="-1"></a>           <span class="at">group.cols =</span> col.hull)</span></code></pre></div>
<p><img src="supp_files/figure-html/pca.plotsiteraw-1.png" width="100%" /></p>
</div>
<div id="procrustes-anova-1" class="section level2 hasAnchor" number="2.4">
<h2><span class="header-section-number">2.4</span> Procrustes ANOVA<a href="gm---siteraw-material.html#procrustes-anova-1" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>To assess whether shape and size differ by <code>size class</code>, Procrustes ANOVAs <span class="citation">(<a href="#ref-RN1749" role="doc-biblioref">Goodall 1991</a>)</span> were run that enlist effect-sizes (zscores) computed as standard deviates of the generated sampling distributions <span class="citation">(<a href="#ref-RN1756" role="doc-biblioref">Collyer, Sekora, and Adams 2015</a>)</span>. A residual randomization permutation procedure (RRPP; n = 10,000 permutations) was used for all Procrustes ANOVAs <span class="citation">(<a href="#ref-RN1655" role="doc-biblioref">Adams and Collyer 2015</a>; <a href="#ref-RN11775" role="doc-biblioref">Collyer and Adams 2018</a>)</span>, which has higher statistical power and a greater ability to identify patterns in the data should they be present <span class="citation">(<a href="#ref-RN1719" role="doc-biblioref">Anderson and Ter Braak 2003</a>)</span>.</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="gm---siteraw-material.html#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="co"># ANOVA ----</span></span>
<span id="cb18-2"><a href="gm---siteraw-material.html#cb18-2" aria-hidden="true" tabindex="-1"></a><span class="co"># shape</span></span>
<span id="cb18-3"><a href="gm---siteraw-material.html#cb18-3" aria-hidden="true" tabindex="-1"></a>fit.sh <span class="ot">&lt;-</span> <span class="fu">procD.lm</span>(shape <span class="sc">~</span> site,</span>
<span id="cb18-4"><a href="gm---siteraw-material.html#cb18-4" aria-hidden="true" tabindex="-1"></a>                   <span class="at">data =</span> gdf,</span>
<span id="cb18-5"><a href="gm---siteraw-material.html#cb18-5" aria-hidden="true" tabindex="-1"></a>                   <span class="at">print.progress =</span> <span class="cn">FALSE</span>,</span>
<span id="cb18-6"><a href="gm---siteraw-material.html#cb18-6" aria-hidden="true" tabindex="-1"></a>                   <span class="at">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb18-7"><a href="gm---siteraw-material.html#cb18-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb18-8"><a href="gm---siteraw-material.html#cb18-8" aria-hidden="true" tabindex="-1"></a><span class="co"># shape as a function of site/raw material?</span></span>
<span id="cb18-9"><a href="gm---siteraw-material.html#cb18-9" aria-hidden="true" tabindex="-1"></a><span class="fu">anova</span>(fit.sh)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##           Df       SS        MS     Rsq      F       Z Pr(&gt;F)
## site       1 0.005626 0.0056261 0.02643 0.7602 0.27634 0.4022
## Residuals 28 0.207216 0.0074006 0.97357                      
## Total     29 0.212842                                        
## 
## Call: procD.lm(f1 = shape ~ site, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="gm---siteraw-material.html#cb20-1" aria-hidden="true" tabindex="-1"></a><span class="co"># size</span></span>
<span id="cb20-2"><a href="gm---siteraw-material.html#cb20-2" aria-hidden="true" tabindex="-1"></a>fit.sz <span class="ot">&lt;-</span> <span class="fu">procD.lm</span>(size <span class="sc">~</span> site,</span>
<span id="cb20-3"><a href="gm---siteraw-material.html#cb20-3" aria-hidden="true" tabindex="-1"></a>                   <span class="at">data =</span> gdf,</span>
<span id="cb20-4"><a href="gm---siteraw-material.html#cb20-4" aria-hidden="true" tabindex="-1"></a>                   <span class="at">print.progress =</span> <span class="cn">FALSE</span>,</span>
<span id="cb20-5"><a href="gm---siteraw-material.html#cb20-5" aria-hidden="true" tabindex="-1"></a>                   <span class="at">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb20-6"><a href="gm---siteraw-material.html#cb20-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-7"><a href="gm---siteraw-material.html#cb20-7" aria-hidden="true" tabindex="-1"></a><span class="co"># size as a function of site/raw material?</span></span>
<span id="cb20-8"><a href="gm---siteraw-material.html#cb20-8" aria-hidden="true" tabindex="-1"></a><span class="fu">anova</span>(fit.sz)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##           Df      SS     MS     Rsq      F      Z Pr(&gt;F)    
## site       1  708.16 708.16 0.36243 15.917 2.9472  8e-04 ***
## Residuals 28 1245.78  44.49 0.63757                         
## Total     29 1953.95                                        
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = size ~ site, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
</div>
<div id="morphological-disparity" class="section level2 hasAnchor" number="2.5">
<h2><span class="header-section-number">2.5</span> Morphological Disparity<a href="gm---siteraw-material.html#morphological-disparity" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The trajectory analysis was followed by an analysis of morphological disparity <span class="citation">(<a href="#ref-RN11107" role="doc-biblioref">Collyer and Adams 2020</a>; <a href="#ref-RN7041" role="doc-biblioref">Foote 1993</a>; <a href="#ref-RN5694" role="doc-biblioref">Zelditch et al. 2004</a>)</span> to identify potential differences in morphological diversity between size classes.</p>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb22-1"><a href="gm---siteraw-material.html#cb22-1" aria-hidden="true" tabindex="-1"></a><span class="co"># morphological disparity ----</span></span>
<span id="cb22-2"><a href="gm---siteraw-material.html#cb22-2" aria-hidden="true" tabindex="-1"></a><span class="co"># do any of the groups display greater shape variation among </span></span>
<span id="cb22-3"><a href="gm---siteraw-material.html#cb22-3" aria-hidden="true" tabindex="-1"></a><span class="co"># individuals relative to the other group?</span></span>
<span id="cb22-4"><a href="gm---siteraw-material.html#cb22-4" aria-hidden="true" tabindex="-1"></a><span class="fu">morphol.disparity</span>(shape <span class="sc">~</span> site,</span>
<span id="cb22-5"><a href="gm---siteraw-material.html#cb22-5" aria-hidden="true" tabindex="-1"></a>                  <span class="at">groups =</span> qdata<span class="sc">$</span>site, </span>
<span id="cb22-6"><a href="gm---siteraw-material.html#cb22-6" aria-hidden="true" tabindex="-1"></a>                  <span class="at">data =</span> gdf, </span>
<span id="cb22-7"><a href="gm---siteraw-material.html#cb22-7" aria-hidden="true" tabindex="-1"></a>                  <span class="at">print.progress =</span> <span class="cn">FALSE</span>, </span>
<span id="cb22-8"><a href="gm---siteraw-material.html#cb22-8" aria-hidden="true" tabindex="-1"></a>                  <span class="at">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
<pre><code>## 
## Call:
## morphol.disparity(f1 = shape ~ site, groups = qdata$site, iter = 9999,  
##     data = gdf, print.progress = FALSE) 
## 
## 
## 
## Randomized Residual Permutation Procedure Used
## 10000 Permutations
## 
## Procrustes variances for defined groups
##   Jowell Farm Millsap Cache 
##   0.011094010   0.004483247 
## 
## 
## Pairwise absolute differences between variances
##               Jowell Farm Millsap Cache
## Jowell Farm   0.000000000   0.006610763
## Millsap Cache 0.006610763   0.000000000
## 
## 
## P-Values
##               Jowell Farm Millsap Cache
## Jowell Farm        1.0000        0.0648
## Millsap Cache      0.0648        1.0000</code></pre>
<div class="sourceCode" id="cb24"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb24-1"><a href="gm---siteraw-material.html#cb24-1" aria-hidden="true" tabindex="-1"></a><span class="co"># do any of the groups display greater size variation among </span></span>
<span id="cb24-2"><a href="gm---siteraw-material.html#cb24-2" aria-hidden="true" tabindex="-1"></a><span class="co"># individuals relative to the other group?</span></span>
<span id="cb24-3"><a href="gm---siteraw-material.html#cb24-3" aria-hidden="true" tabindex="-1"></a><span class="fu">morphol.disparity</span>(size <span class="sc">~</span> site,</span>
<span id="cb24-4"><a href="gm---siteraw-material.html#cb24-4" aria-hidden="true" tabindex="-1"></a>                  <span class="at">groups =</span> qdata<span class="sc">$</span>site, </span>
<span id="cb24-5"><a href="gm---siteraw-material.html#cb24-5" aria-hidden="true" tabindex="-1"></a>                  <span class="at">data =</span> gdf, </span>
<span id="cb24-6"><a href="gm---siteraw-material.html#cb24-6" aria-hidden="true" tabindex="-1"></a>                  <span class="at">print.progress =</span> <span class="cn">FALSE</span>, </span>
<span id="cb24-7"><a href="gm---siteraw-material.html#cb24-7" aria-hidden="true" tabindex="-1"></a>                  <span class="at">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
<pre><code>## 
## Call:
## morphol.disparity(f1 = size ~ site, groups = qdata$site, iter = 9999,  
##     data = gdf, print.progress = FALSE) 
## 
## 
## 
## Randomized Residual Permutation Procedure Used
## 10000 Permutations
## 
## Procrustes variances for defined groups
##   Jowell Farm Millsap Cache 
##      16.18733      56.19584 
## 
## 
## Pairwise absolute differences between variances
##               Jowell Farm Millsap Cache
## Jowell Farm       0.00000      40.00852
## Millsap Cache    40.00852       0.00000
## 
## 
## P-Values
##               Jowell Farm Millsap Cache
## Jowell Farm        1.0000        0.1366
## Millsap Cache      0.1366        1.0000</code></pre>

</div>
</div>
<h3>References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-RN1655" class="csl-entry">
Adams, Dean C., and Michael L. Collyer. 2015. <span>“<span class="nocase">Permutation Tests for Phylogenetic Comparative Analyses of High-Dimensional Shape Data: What you Shuffle Matters</span>.”</span> <em>Evolution</em> 69 (3): 823–29. <a href="https://doi.org/10.1111/evo.12596">https://doi.org/10.1111/evo.12596</a>.
</div>
<div id="ref-RN11530" class="csl-entry">
Adams, Dean C., Michael L. Collyer, Antigoni Kaliontzopoulou, and Emma Sherratt. 2017. <span>“<span class="nocase">Package geomorph: Geometric Morphometric Analyses of Landmark Data. R package version 3.0.5</span>.”</span> <a href="http://geomorphr.github.io/geomorph/">http://geomorphr.github.io/geomorph/</a>.
</div>
<div id="ref-RN1774" class="csl-entry">
Adams, Dean C., and Erik Otarola-Castillo. 2013. <span>“<span class="nocase">geomorph: An R Package for the Collection and Analysis of Geometric Morphometric Shape Data</span>.”</span> <em>Methods in Ecology and Evolution</em> 4 (4): 393–99. <a href="https://doi.org/10.1111/2041-210x.12035">https://doi.org/10.1111/2041-210x.12035</a>.
</div>
<div id="ref-RN1719" class="csl-entry">
Anderson, M. J., and C. J. F. Ter Braak. 2003. <span>“<span class="nocase">Permutation Tests for Multi-Factoral Analysis of Variance</span>.”</span> <em>Journal of Statistical Computation and Simulation</em> 73 (2): 85–113. <a href="https://doi.org/10.1080=0094965021000015558">https://doi.org/10.1080=0094965021000015558</a>.
</div>
<div id="ref-RN9565" class="csl-entry">
Baken, Erica K., Michael L. Collyer, Antigoni Kaliontzopoulou, and Dean C. Adams. 2021. <span>“<span class="nocase">geomorph v4.0 and gmShiny: Enhanced analytics and a new graphical interface for a comprehensive morphometric experience</span>.”</span> Journal Article. <em>Methods in Ecology and Evolution</em>. <a href="https://doi.org/10.1111/2041-210x.13723">https://doi.org/10.1111/2041-210x.13723</a>.
</div>
<div id="ref-RN11775" class="csl-entry">
Collyer, Michael L., and Dean C. Adams. 2018. <span>“<span class="nocase">RRPP: An R Package for Fitting Linear Models to High-Dimensional Data using Residual Randomization</span>.”</span> <em>Methods in Ecology and Evolution</em> 9 (7): 1772–79. https://doi.org/<a href="https://doi.org/10.1111/2041-210X.13029">https://doi.org/10.1111/2041-210X.13029</a>.
</div>
<div id="ref-RN11107" class="csl-entry">
———. 2020. <span>“Phylogenetically Aligned Component Analysis.”</span> Journal Article. <em>Methods in Ecology and Evolution</em> 12 (2): 359–72. <a href="https://doi.org/10.1111/2041-210x.13515">https://doi.org/10.1111/2041-210x.13515</a>.
</div>
<div id="ref-RN1756" class="csl-entry">
Collyer, Michael L., David J. Sekora, and Dean C. Adams. 2015. <span>“<span class="nocase">A Method for Analysis of Phenotypic Change for Phenotypes Described by High-Dimensional Data</span>.”</span> <em>Heredity</em> 115 (4): 357–65. <a href="https://doi.org/10.1038/hdy.2014.75">https://doi.org/10.1038/hdy.2014.75</a>.
</div>
<div id="ref-RN7041" class="csl-entry">
Foote, M. 1993. <span>“Contributions of Individual Taxa to Overall Morphological Disparity.”</span> Journal Article. <em>Paleobiology</em> 19 (4): 403–19.
</div>
<div id="ref-RN1749" class="csl-entry">
Goodall, Colin. 1991. <span>“<span class="nocase">Procrustes Methods in the Statistical Analysis of Shape</span>.”</span> <em>Journal of the Royal Statistical Society. Series B (Methodological)</em> 53 (2): 285–339.
</div>
<div id="ref-RN11564" class="csl-entry">
Gower, J. C. 1975. <span>“<span>Generalized Procrustes Analysis</span>.”</span> <em>Psychometrika</em> 40 (1): 33–51. https://doi.org/<a href="https://doi.org/10.1007/BF02291478">https://doi.org/10.1007/BF02291478</a>.
</div>
<div id="ref-RN1746" class="csl-entry">
Jolliffe, Ian T. 2002. <em>Principal Component Analysis</em>. New York: Springer.
</div>
<div id="ref-RN11622" class="csl-entry">
Kendall, David G. 1981. <span>“<span class="nocase">The Statistics of Shape</span>.”</span> In <em>Interpreting Multivariate Data</em>, edited by V. Barnett, 75–80. New York: Wiley.
</div>
<div id="ref-RN11623" class="csl-entry">
———. 1984. <span>“<span class="nocase">Shape Manifolds, Procrustean Metrics, and Complex Projective Spaces</span>.”</span> <em>Bulletin of the London Mathematical Society</em> 16 (2): 81–121. <a href="https://doi.org/10.1112/blms/16.2.81">https://doi.org/10.1112/blms/16.2.81</a>.
</div>
<div id="ref-RN1731" class="csl-entry">
Klingenberg, Christian Peter. 2013. <span>“<span class="nocase">Visualizations in Geometric Morphometrics: How to Read and How to Make Graphs Showing Shape Changes</span>.”</span> <em>Hystrix</em> 24 (1): 15–24. https://doi.org/<a href="http://dx.doi.org/10.4404/hystrix-24.1-7691">http://dx.doi.org/10.4404/hystrix-24.1-7691</a>.
</div>
<div id="ref-R" class="csl-entry">
R Core Development Team,. 2022. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="http://www.R-project.org/">http://www.R-project.org/</a>.
</div>
<div id="ref-RN1646" class="csl-entry">
Rohlf, F. James. 1999. <span>“<span class="nocase">Shape Statistics: Procrustes Superimpositions and Tangent Spaces</span>.”</span> <em>Journal of Classification</em> 16 (2): 197–223. <a href="https://doi.org/10.1007/s003579900054">https://doi.org/10.1007/s003579900054</a>.
</div>
<div id="ref-RN478" class="csl-entry">
Rohlf, F. James, and Dennis Slice. 1990. <span>“<span class="nocase">Extensions of the Procrustes Method for the Optimal Superimposition of Landmarks</span>.”</span> <em>Systematic Zoology</em> 39 (1): 40–59. <a href="https://doi.org/10.2307/2992207">https://doi.org/10.2307/2992207</a>.
</div>
<div id="ref-RN9551" class="csl-entry">
———. 2021b. <span>“<span class="nocase">Shape as a function of time + raw material + burial context? An exploratory analysis of Perdiz arrow points from the ancestral Caddo area of the American Southeast</span>.”</span> Journal Article. <em>Journal of Archaeological Science: Reports</em> 37: 102916. <a href="https://doi.org/10.1016/j.jasrep.2021.102916">https://doi.org/10.1016/j.jasrep.2021.102916</a>.
</div>
<div id="ref-RN11801" class="csl-entry">
Selden Jr., Robert Z. 2018a. <span>“A Preliminary Study of Smithport Plain Bottle Morphology in the Southern Caddo Area.”</span> <em>Bulletin of the Texas Archeological Society</em> 89: 63–89.
</div>
<div id="ref-RN11782" class="csl-entry">
———. 2018b. <span>“<span class="nocase">Ceramic Morphological Organisation in the Southern Caddo Area: Quiddity of Shape for Hickory Engraved Bottles</span>.”</span> <em>Journal of Archaeological Science: Reports</em> 21: 884–96. <a href="https://doi.org/10.1016/j.jasrep.2018.08.045">https://doi.org/10.1016/j.jasrep.2018.08.045</a>.
</div>
<div id="ref-RN11716" class="csl-entry">
———. 2019. <span>“<span class="nocase">Ceramic Morphological Organisation in the Southern Caddo Area: The Clarence H. Webb Collections</span>.”</span> <em>Journal of Cultural Heritage</em> 35: 41–55. https://doi.org/<a href="https://doi.org/10.1016/j.culher.2018.07.002">https://doi.org/10.1016/j.culher.2018.07.002</a>.
</div>
<div id="ref-RN8312" class="csl-entry">
———. 2021. <span>“Louisiana Limitrophe: An Iterative Morphological Exegesis of Caddo Bottle and Biface Production.”</span> Book Section. In <em>Ancestral Caddo Ceramic Traditions</em>, edited by Duncan P. McKinnon, Jeffrey S. Girard, and Timothy K. Perttula, 258–76. Baton Rouge: LSU Press.
</div>
<div id="ref-RN21001" class="csl-entry">
Selden Jr., Robert Z., John E. Dockall, and Morgane Dubied. 2020. <span>“A Quantitative Assessment of Intraspecific Morphological Variation in Gahagan Bifaces from the Southern Caddo Area and Central Texas.”</span> <em>Southeastern Archaeology</em> 39 (2): 125–45. <a href="https://doi.org/10.1080/0734578x.2020.1744416">https://doi.org/10.1080/0734578x.2020.1744416</a>.
</div>
<div id="ref-RN11783" class="csl-entry">
Selden Jr., Robert Z., John E. Dockall, and Harry J. Shafer. 2018. <span>“<span class="nocase">Lithic Morphological Organisation: Gahagan Bifaces from the Southern Caddo Area</span>.”</span> <em>Digital Applications in Archaeology and Cultural Heritage</em> 10: e00080. <a href="https://doi.org/10.1016/j.daach.2018.e00080">https://doi.org/10.1016/j.daach.2018.e00080</a>.
</div>
<div id="ref-RN11264" class="csl-entry">
Selden, Robert Z. 2022. <span>“Morphologically Similar, but Regionally Distinct: Perdiz Arrow Points from Caddo Burial Contexts in the American Southeast.”</span> Journal Article. <em>Lithic Technology</em> (in press). <a href="https://doi.org/10.1080/01977261.2022.2095492">https://doi.org/10.1080/01977261.2022.2095492</a>.
</div>
<div id="ref-RN479" class="csl-entry">
Sherratt, Emma, David J. Gower, Christian Peter Klingenberg, and Mark Wilkinson. 2014. <span>“<span class="nocase">Evolution of Cranial Shape in Caecilians (Amphibia: Gymnophiona)</span>.”</span> <em>Evolutionary Biology</em> 41 (4): 528–45. https://doi.org/<a href="https://doi.org/10.1007/s11692-014-9287-2">https://doi.org/10.1007/s11692-014-9287-2</a>.
</div>
<div id="ref-RN11563" class="csl-entry">
Slice, Dennis E. 2001. <span>“<span class="nocase">Landmark Coordinates Aligned by Procrustes Analysis Do Not Lie in Kendall’s Shape Space</span>.”</span> <em>Systematic Biology</em> 50 (1): 141–49. <a href="https://doi.org/10.1080/10635150119110">https://doi.org/10.1080/10635150119110</a>.
</div>
<div id="ref-RN5694" class="csl-entry">
Zelditch, Miriam Leah, Donald L. Swiderski, H. David Sheets, and William L. Fink. 2004. <em>Geometric Morphometrics for Biologists : A Primer</em>. Book. Burlington: Elsevier Science. <a href="http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308">http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="define-size-classes.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="gm---size-class.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "night",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["supp.pdf", "supp.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
