<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 GM - Size Class | Caddo retouch results in divergent morphology for elliptical bifaces from sites in the American Southeast</title>
  <meta name="description" content="Pilot study of biface morphology." />
  <meta name="generator" content="bookdown 0.27 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 GM - Size Class | Caddo retouch results in divergent morphology for elliptical bifaces from sites in the American Southeast" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Pilot study of biface morphology." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 GM - Size Class | Caddo retouch results in divergent morphology for elliptical bifaces from sites in the American Southeast" />
  
  <meta name="twitter:description" content="Pilot study of biface morphology." />
  

<meta name="author" content="Robert Z. Selden, Jr., and John E. Dockall" />


<meta name="date" content="2022-08-04" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="gm---siteraw-material.html"/>
<link rel="next" href="references.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="custom.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">jowell.1</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a>
<ul>
<li class="chapter" data-level="0.1" data-path="index.html"><a href="index.html#primary-findings"><i class="fa fa-check"></i><b>0.1</b> Primary findings</a></li>
<li class="chapter" data-level="0.2" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i><b>0.2</b> Acknowledgments</a></li>
<li class="chapter" data-level="0.3" data-path="index.html"><a href="index.html#funding"><i class="fa fa-check"></i><b>0.3</b> Funding</a></li>
<li class="chapter" data-level="0.4" data-path="index.html"><a href="index.html#data-management"><i class="fa fa-check"></i><b>0.4</b> Data management</a></li>
<li class="chapter" data-level="0.5" data-path="index.html"><a href="index.html#colophon"><i class="fa fa-check"></i><b>0.5</b> Colophon</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="define-size-classes.html"><a href="define-size-classes.html"><i class="fa fa-check"></i><b>1</b> Define size classes</a>
<ul>
<li class="chapter" data-level="1.1" data-path="define-size-classes.html"><a href="define-size-classes.html#load-packages-data"><i class="fa fa-check"></i><b>1.1</b> Load packages + data</a></li>
<li class="chapter" data-level="1.2" data-path="define-size-classes.html"><a href="define-size-classes.html#generalised-procrustes-analysis"><i class="fa fa-check"></i><b>1.2</b> Generalised Procrustes Analysis</a></li>
<li class="chapter" data-level="1.3" data-path="define-size-classes.html"><a href="define-size-classes.html#assign-size-class"><i class="fa fa-check"></i><b>1.3</b> Assign size class</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html"><i class="fa fa-check"></i><b>2</b> GM - Site/Raw Material</a>
<ul>
<li class="chapter" data-level="2.1" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html#load-packages-data-1"><i class="fa fa-check"></i><b>2.1</b> Load packages + data</a></li>
<li class="chapter" data-level="2.2" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html#generalised-procrustes-analysis-1"><i class="fa fa-check"></i><b>2.2</b> Generalised Procrustes Analysis</a></li>
<li class="chapter" data-level="2.3" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html#principal-components-analysis"><i class="fa fa-check"></i><b>2.3</b> Principal Components Analysis</a></li>
<li class="chapter" data-level="2.4" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html#procrustes-anova"><i class="fa fa-check"></i><b>2.4</b> Procrustes ANOVA</a></li>
<li class="chapter" data-level="2.5" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html#morphological-disparity"><i class="fa fa-check"></i><b>2.5</b> Morphological Disparity</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="gm---size-class.html"><a href="gm---size-class.html"><i class="fa fa-check"></i><b>3</b> GM - Size Class</a>
<ul>
<li class="chapter" data-level="3.1" data-path="gm---size-class.html"><a href="gm---size-class.html#load-packages-data-2"><i class="fa fa-check"></i><b>3.1</b> Load packages + data</a></li>
<li class="chapter" data-level="3.2" data-path="gm---size-class.html"><a href="gm---size-class.html#generalised-procrustes-analysis-2"><i class="fa fa-check"></i><b>3.2</b> Generalised Procrustes Analysis</a></li>
<li class="chapter" data-level="3.3" data-path="gm---size-class.html"><a href="gm---size-class.html#principal-components-analysis-1"><i class="fa fa-check"></i><b>3.3</b> Principal Components Analysis</a></li>
<li class="chapter" data-level="3.4" data-path="gm---size-class.html"><a href="gm---size-class.html#procrustes-anova-1"><i class="fa fa-check"></i><b>3.4</b> Procrustes ANOVA</a></li>
<li class="chapter" data-level="3.5" data-path="gm---size-class.html"><a href="gm---size-class.html#trajectory-analysis"><i class="fa fa-check"></i><b>3.5</b> Trajectory analysis</a></li>
<li class="chapter" data-level="3.6" data-path="gm---size-class.html"><a href="gm---size-class.html#morphological-disparity-1"><i class="fa fa-check"></i><b>3.6</b> Morphological Disparity</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Caddo retouch results in divergent morphology for elliptical bifaces from sites in the American Southeast</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="gm---size-class" class="section level1 hasAnchor" number="3">
<h1><span class="header-section-number">Chapter 3</span> GM - Size Class<a href="gm---size-class.html#gm---size-class" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Two <code>size classes</code> (large/small) were used to examine whether the morphology of the elliptical bifaces from the Millsap Cache and Jowell Farm remain morphologically stable, or express morphological variation. Variability in elliptical biface morphology potentially provides evidence associated with differential resharpening trajectories.</p>
<p>This analysis follows recent analyses of Perdiz arrow points (<a href="https://github.com/seldenlab/perdiz3/blob/main/ms/perdiz3/perdiz3.pdf">Selden and Dockall, in review</a>) and Gahagan biface morphology across the southern Caddo area and central Texas <span class="citation">(<a href="#ref-RN11783" role="doc-biblioref">Selden Jr., Dockall, and Shafer 2018</a>; <a href="#ref-RN21001" role="doc-biblioref">Selden Jr., Dockall, and Dubied 2020</a>; <a href="#ref-RN11264" role="doc-biblioref">Selden 2022</a>)</span>, where Perdiz arrow points and Gahagan bifaces have been found to differ in shape across the same geography as Caddo bottles <span class="citation">(<a href="#ref-RN11801" role="doc-biblioref">Selden Jr. 2018a</a>, <a href="#ref-RN11782" role="doc-biblioref">2018b</a>, <a href="#ref-RN11716" role="doc-biblioref">2019</a>, <a href="#ref-RN8312" role="doc-biblioref">2021</a>)</span>. This analysis also follows another geometric morphometric analysis in which Perdiz arrow points from the ancestral Caddo area were found to differ as a function of <em>time</em>, <em>raw material</em>, and <em>burial context</em> <span class="citation">(<a href="#ref-RN9551" role="doc-biblioref">Selden Jr et al. 2021b</a>)</span>.</p>
<div id="load-packages-data-2" class="section level2 hasAnchor" number="3.1">
<h2><span class="header-section-number">3.1</span> Load packages + data<a href="gm---size-class.html#load-packages-data-2" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb33-1"><a href="gm---size-class.html#cb33-1" aria-hidden="true" tabindex="-1"></a><span class="co"># load required analysis packages</span></span>
<span id="cb33-2"><a href="gm---size-class.html#cb33-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(here)</span>
<span id="cb33-3"><a href="gm---size-class.html#cb33-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(StereoMorph)</span>
<span id="cb33-4"><a href="gm---size-class.html#cb33-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(geomorph)</span>
<span id="cb33-5"><a href="gm---size-class.html#cb33-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb33-6"><a href="gm---size-class.html#cb33-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(wesanderson)</span>
<span id="cb33-7"><a href="gm---size-class.html#cb33-7" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb33-8"><a href="gm---size-class.html#cb33-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-9"><a href="gm---size-class.html#cb33-9" aria-hidden="true" tabindex="-1"></a><span class="co"># read data and define number of sLMs ----</span></span>
<span id="cb33-10"><a href="gm---size-class.html#cb33-10" aria-hidden="true" tabindex="-1"></a>shapes <span class="ot">&lt;-</span> <span class="fu">readShapes</span>(<span class="st">&quot;shapes&quot;</span>)</span>
<span id="cb33-11"><a href="gm---size-class.html#cb33-11" aria-hidden="true" tabindex="-1"></a>shapesGM <span class="ot">&lt;-</span> <span class="fu">readland.shapes</span>(shapes, <span class="at">nCurvePts =</span> <span class="fu">c</span>(<span class="dv">75</span>, <span class="dv">75</span>))</span>
<span id="cb33-12"><a href="gm---size-class.html#cb33-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-13"><a href="gm---size-class.html#cb33-13" aria-hidden="true" tabindex="-1"></a><span class="co"># read qualitative data</span></span>
<span id="cb33-14"><a href="gm---size-class.html#cb33-14" aria-hidden="true" tabindex="-1"></a>qdata <span class="ot">&lt;-</span> <span class="fu">read.csv</span>(<span class="st">&quot;qdata.merged.csv&quot;</span>, </span>
<span id="cb33-15"><a href="gm---size-class.html#cb33-15" aria-hidden="true" tabindex="-1"></a>                  <span class="at">header =</span> <span class="cn">TRUE</span>, </span>
<span id="cb33-16"><a href="gm---size-class.html#cb33-16" aria-hidden="true" tabindex="-1"></a>                  <span class="at">row.names =</span> <span class="dv">1</span>)</span></code></pre></div>
</div>
<div id="generalised-procrustes-analysis-2" class="section level2 hasAnchor" number="3.2">
<h2><span class="header-section-number">3.2</span> Generalised Procrustes Analysis<a href="gm---size-class.html#generalised-procrustes-analysis-2" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Landmark data were aligned to a global coordinate system <span class="citation">(<a href="#ref-RN11622" role="doc-biblioref">Kendall 1981</a>, <a href="#ref-RN11623" role="doc-biblioref">1984</a>; <a href="#ref-RN11563" role="doc-biblioref">Slice 2001</a>)</span>, achieved through generalized Procrustes superimposition <span class="citation">(<a href="#ref-RN478" role="doc-biblioref">Rohlf and Slice 1990</a>)</span> performed in R 4.1.3 <span class="citation">(<a href="#ref-R" role="doc-biblioref">R Core Development Team, 2022</a>)</span> using the <code>geomorph</code> library v. 4.0.3 <span class="citation">(<a href="#ref-RN11530" role="doc-biblioref">Adams et al. 2017</a>; <a href="#ref-RN1774" role="doc-biblioref">Adams and Otarola-Castillo 2013</a>; <a href="#ref-RN9565" role="doc-biblioref">Baken et al. 2021</a>)</span>. Procrustes superimposition translates, scales, and rotates the coordinate data to allow for comparisons among objects <span class="citation">(<a href="#ref-RN11564" role="doc-biblioref">Gower 1975</a>; <a href="#ref-RN478" role="doc-biblioref">Rohlf and Slice 1990</a>)</span>. The <code>geomorph</code> package uses a partial Procrustes superimposition that projects the aligned specimens into tangent space subsequent to alignment in preparation for the use of multivariate methods that assume linear space <span class="citation">(<a href="#ref-RN1646" role="doc-biblioref">Rohlf 1999</a>; <a href="#ref-RN11563" role="doc-biblioref">Slice 2001</a>)</span>.</p>
<div class="sourceCode" id="cb34"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb34-1"><a href="gm---size-class.html#cb34-1" aria-hidden="true" tabindex="-1"></a><span class="co"># gpa ----</span></span>
<span id="cb34-2"><a href="gm---size-class.html#cb34-2" aria-hidden="true" tabindex="-1"></a>Y.gpa <span class="ot">&lt;-</span> <span class="fu">gpagen</span>(shapesGM, <span class="at">print.progress =</span> <span class="cn">FALSE</span>)</span>
<span id="cb34-3"><a href="gm---size-class.html#cb34-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-4"><a href="gm---size-class.html#cb34-4" aria-hidden="true" tabindex="-1"></a><span class="co"># geomorph data frame ----</span></span>
<span id="cb34-5"><a href="gm---size-class.html#cb34-5" aria-hidden="true" tabindex="-1"></a>gdf <span class="ot">&lt;-</span> <span class="fu">geomorph.data.frame</span>(<span class="at">shape =</span> Y.gpa<span class="sc">$</span>coords, </span>
<span id="cb34-6"><a href="gm---size-class.html#cb34-6" aria-hidden="true" tabindex="-1"></a>                           <span class="at">size =</span> Y.gpa<span class="sc">$</span>Csize,</span>
<span id="cb34-7"><a href="gm---size-class.html#cb34-7" aria-hidden="true" tabindex="-1"></a>                           <span class="at">site =</span> qdata<span class="sc">$</span>site,</span>
<span id="cb34-8"><a href="gm---size-class.html#cb34-8" aria-hidden="true" tabindex="-1"></a>                           <span class="at">size.class =</span> qdata<span class="sc">$</span>size,</span>
<span id="cb34-9"><a href="gm---size-class.html#cb34-9" aria-hidden="true" tabindex="-1"></a>                           <span class="at">merged =</span> qdata<span class="sc">$</span>merged)</span>
<span id="cb34-10"><a href="gm---size-class.html#cb34-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-11"><a href="gm---size-class.html#cb34-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-12"><a href="gm---size-class.html#cb34-12" aria-hidden="true" tabindex="-1"></a><span class="co"># add centroid size to qdata</span></span>
<span id="cb34-13"><a href="gm---size-class.html#cb34-13" aria-hidden="true" tabindex="-1"></a>qdata<span class="sc">$</span>csz <span class="ot">&lt;-</span> Y.gpa<span class="sc">$</span>Csize</span></code></pre></div>
</div>
<div id="principal-components-analysis-1" class="section level2 hasAnchor" number="3.3">
<h2><span class="header-section-number">3.3</span> Principal Components Analysis<a href="gm---size-class.html#principal-components-analysis-1" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Principal components analysis <span class="citation">(<a href="#ref-RN1746" role="doc-biblioref">Jolliffe 2002</a>)</span> was used to visualise shape variation among the Perdiz arrow points. The shape changes described by each principal axis are commonly visualized using thin-plate spline warping of a reference 3D mesh <span class="citation">(<a href="#ref-RN1731" role="doc-biblioref">Klingenberg 2013</a>; <a href="#ref-RN479" role="doc-biblioref">Sherratt et al. 2014</a>)</span>.</p>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb35-1"><a href="gm---size-class.html#cb35-1" aria-hidden="true" tabindex="-1"></a><span class="co"># principal components analysis ----</span></span>
<span id="cb35-2"><a href="gm---size-class.html#cb35-2" aria-hidden="true" tabindex="-1"></a>pca<span class="ot">&lt;-</span><span class="fu">gm.prcomp</span>(Y.gpa<span class="sc">$</span>coords)</span>
<span id="cb35-3"><a href="gm---size-class.html#cb35-3" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(pca)</span></code></pre></div>
<pre><code>## 
## Ordination type: Principal Component Analysis 
## Centering by OLS mean
## Orthogonal projection of OLS residuals
## Number of observations: 30 
## Number of vectors 29 
## 
## Importance of Components:
##                               Comp1        Comp2        Comp3        Comp4        Comp5        Comp6        Comp7        Comp8
## Eigenvalues            0.0008497355 0.0004220093 0.0003408303 6.536069e-05 2.677863e-05 1.821953e-05 1.256666e-05 7.525419e-06
## Proportion of Variance 0.4831142502 0.2399319701 0.1937779309 3.716060e-02 1.522490e-02 1.035865e-02 7.144731e-03 4.278551e-03
## Cumulative Proportion  0.4831142502 0.7230462202 0.9168241511 9.539847e-01 9.692096e-01 9.795683e-01 9.867130e-01 9.909916e-01
##                               Comp9       Comp10       Comp11       Comp12       Comp13       Comp14       Comp15       Comp16
## Eigenvalues            4.853207e-06 4.233497e-06 1.650486e-06 1.474448e-06 1.064822e-06 6.037073e-07 5.398387e-07 3.736349e-07
## Proportion of Variance 2.759275e-03 2.406940e-03 9.383783e-04 8.382926e-04 6.054009e-04 3.432357e-04 3.069235e-04 2.124289e-04
## Cumulative Proportion  9.937509e-01 9.961578e-01 9.970962e-01 9.979345e-01 9.985399e-01 9.988831e-01 9.991900e-01 9.994025e-01
##                              Comp17       Comp18       Comp19       Comp20       Comp21       Comp22       Comp23       Comp24
## Eigenvalues            2.595297e-07 1.715361e-07 1.497939e-07 8.555405e-08 8.064810e-08 6.168772e-08 5.720117e-08 4.476731e-08
## Proportion of Variance 1.475547e-04 9.752627e-05 8.516481e-05 4.864147e-05 4.585221e-05 3.507235e-05 3.252153e-05 2.545231e-05
## Cumulative Proportion  9.995500e-01 9.996475e-01 9.997327e-01 9.997813e-01 9.998272e-01 9.998623e-01 9.998948e-01 9.999202e-01
##                              Comp25       Comp26       Comp27       Comp28       Comp29
## Eigenvalues            4.047475e-08 3.310299e-08 2.659745e-08 2.238423e-08 1.771950e-08
## Proportion of Variance 2.301178e-05 1.882059e-05 1.512189e-05 1.272648e-05 1.007436e-05
## Cumulative Proportion  9.999433e-01 9.999621e-01 9.999772e-01 9.999899e-01 1.000000e+00</code></pre>
<div class="sourceCode" id="cb37"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb37-1"><a href="gm---size-class.html#cb37-1" aria-hidden="true" tabindex="-1"></a><span class="co"># set plot parameters</span></span>
<span id="cb37-2"><a href="gm---size-class.html#cb37-2" aria-hidden="true" tabindex="-1"></a>pch.gps <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">4</span>)[<span class="fu">as.factor</span>(qdata<span class="sc">$</span>merged)]</span>
<span id="cb37-3"><a href="gm---size-class.html#cb37-3" aria-hidden="true" tabindex="-1"></a>col.gps <span class="ot">&lt;-</span> <span class="fu">wes_palette</span>(<span class="st">&quot;Moonrise2&quot;</span>)[<span class="fu">as.factor</span>(qdata<span class="sc">$</span>merged)]</span>
<span id="cb37-4"><a href="gm---size-class.html#cb37-4" aria-hidden="true" tabindex="-1"></a>col.hull <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">&quot;#CCC591&quot;</span>, <span class="st">&quot;#29211F&quot;</span>, <span class="st">&quot;#798E87&quot;</span>, <span class="st">&quot;#C27D38&quot;</span>)</span>
<span id="cb37-5"><a href="gm---size-class.html#cb37-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-6"><a href="gm---size-class.html#cb37-6" aria-hidden="true" tabindex="-1"></a><span class="co"># plot pca by merged</span></span>
<span id="cb37-7"><a href="gm---size-class.html#cb37-7" aria-hidden="true" tabindex="-1"></a>pc.plot <span class="ot">&lt;-</span> <span class="fu">plot</span>(pca, </span>
<span id="cb37-8"><a href="gm---size-class.html#cb37-8" aria-hidden="true" tabindex="-1"></a>                <span class="at">asp =</span> <span class="dv">1</span>,</span>
<span id="cb37-9"><a href="gm---size-class.html#cb37-9" aria-hidden="true" tabindex="-1"></a>                <span class="at">pch =</span> pch.gps,</span>
<span id="cb37-10"><a href="gm---size-class.html#cb37-10" aria-hidden="true" tabindex="-1"></a>                <span class="at">col =</span> col.gps)</span>
<span id="cb37-11"><a href="gm---size-class.html#cb37-11" aria-hidden="true" tabindex="-1"></a><span class="fu">shapeHulls</span>(pc.plot, </span>
<span id="cb37-12"><a href="gm---size-class.html#cb37-12" aria-hidden="true" tabindex="-1"></a>           <span class="at">groups =</span> qdata<span class="sc">$</span>merged,</span>
<span id="cb37-13"><a href="gm---size-class.html#cb37-13" aria-hidden="true" tabindex="-1"></a>           <span class="at">group.cols =</span> col.hull)</span></code></pre></div>
<p><img src="supp_files/figure-html/pca.plot-1.png" width="100%" /></p>
</div>
<div id="procrustes-anova-1" class="section level2 hasAnchor" number="3.4">
<h2><span class="header-section-number">3.4</span> Procrustes ANOVA<a href="gm---size-class.html#procrustes-anova-1" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>To assess whether shape and size differ by <code>size class</code>, Procrustes ANOVAs <span class="citation">(<a href="#ref-RN1749" role="doc-biblioref">Goodall 1991</a>)</span> were run that enlist effect-sizes (zscores) computed as standard deviates of the generated sampling distributions <span class="citation">(<a href="#ref-RN1756" role="doc-biblioref">M. L. Collyer, Sekora, and Adams 2015</a>)</span>. A residual randomization permutation procedure (RRPP; n = 10,000 permutations) was used for all Procrustes ANOVAs <span class="citation">(<a href="#ref-RN1655" role="doc-biblioref">Adams and Collyer 2015</a>; <a href="#ref-RN11775" role="doc-biblioref">M. L. Collyer and Adams 2018</a>)</span>, which has higher statistical power and a greater ability to identify patterns in the data should they be present <span class="citation">(<a href="#ref-RN1719" role="doc-biblioref">Anderson and Ter Braak 2003</a>)</span>.</p>
<div class="sourceCode" id="cb38"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb38-1"><a href="gm---size-class.html#cb38-1" aria-hidden="true" tabindex="-1"></a><span class="co"># ANOVA ----</span></span>
<span id="cb38-2"><a href="gm---size-class.html#cb38-2" aria-hidden="true" tabindex="-1"></a><span class="co"># shape</span></span>
<span id="cb38-3"><a href="gm---size-class.html#cb38-3" aria-hidden="true" tabindex="-1"></a>fit.sh <span class="ot">&lt;-</span> <span class="fu">procD.lm</span>(shape <span class="sc">~</span> merged,</span>
<span id="cb38-4"><a href="gm---size-class.html#cb38-4" aria-hidden="true" tabindex="-1"></a>                   <span class="at">data =</span> gdf,</span>
<span id="cb38-5"><a href="gm---size-class.html#cb38-5" aria-hidden="true" tabindex="-1"></a>                   <span class="at">print.progress =</span> <span class="cn">FALSE</span>,</span>
<span id="cb38-6"><a href="gm---size-class.html#cb38-6" aria-hidden="true" tabindex="-1"></a>                   <span class="at">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb38-7"><a href="gm---size-class.html#cb38-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-8"><a href="gm---size-class.html#cb38-8" aria-hidden="true" tabindex="-1"></a><span class="co"># shape as a function of size class?</span></span>
<span id="cb38-9"><a href="gm---size-class.html#cb38-9" aria-hidden="true" tabindex="-1"></a><span class="fu">anova</span>(fit.sh)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##           Df       SS        MS     Rsq      F        Z Pr(&gt;F)
## merged     3 0.003698 0.0012328 0.07251 0.6775 -0.67107 0.7481
## Residuals 26 0.047309 0.0018196 0.92749                       
## Total     29 0.051007                                         
## 
## Call: procD.lm(f1 = shape ~ merged, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb40"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb40-1"><a href="gm---size-class.html#cb40-1" aria-hidden="true" tabindex="-1"></a><span class="co"># size</span></span>
<span id="cb40-2"><a href="gm---size-class.html#cb40-2" aria-hidden="true" tabindex="-1"></a>fit.sz <span class="ot">&lt;-</span> <span class="fu">procD.lm</span>(size <span class="sc">~</span> merged,</span>
<span id="cb40-3"><a href="gm---size-class.html#cb40-3" aria-hidden="true" tabindex="-1"></a>                   <span class="at">data =</span> gdf,</span>
<span id="cb40-4"><a href="gm---size-class.html#cb40-4" aria-hidden="true" tabindex="-1"></a>                   <span class="at">print.progress =</span> <span class="cn">FALSE</span>,</span>
<span id="cb40-5"><a href="gm---size-class.html#cb40-5" aria-hidden="true" tabindex="-1"></a>                   <span class="at">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb40-6"><a href="gm---size-class.html#cb40-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-7"><a href="gm---size-class.html#cb40-7" aria-hidden="true" tabindex="-1"></a><span class="co"># size as a function of size class?</span></span>
<span id="cb40-8"><a href="gm---size-class.html#cb40-8" aria-hidden="true" tabindex="-1"></a><span class="fu">anova</span>(fit.sz)</span></code></pre></div>
<pre><code>## 
## Analysis of Variance, using Residual Randomization
## Permutation procedure: Randomization of null model residuals 
## Number of permutations: 10000 
## Estimation method: Ordinary Least Squares 
## Sums of Squares and Cross-products: Type I 
## Effect sizes (Z) based on F distributions
## 
##           Df     SS     MS     Rsq      F      Z Pr(&gt;F)    
## merged     3 6172.7 2057.6 0.69866 20.094 5.0895  1e-04 ***
## Residuals 26 2662.4  102.4 0.30134                         
## Total     29 8835.1                                        
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Call: procD.lm(f1 = size ~ merged, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
<div class="sourceCode" id="cb42"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb42-1"><a href="gm---size-class.html#cb42-1" aria-hidden="true" tabindex="-1"></a><span class="co"># pairwise comparison of LS means = which differ?</span></span>
<span id="cb42-2"><a href="gm---size-class.html#cb42-2" aria-hidden="true" tabindex="-1"></a>pw.sz <span class="ot">&lt;-</span> <span class="fu">pairwise</span>(fit.sz,</span>
<span id="cb42-3"><a href="gm---size-class.html#cb42-3" aria-hidden="true" tabindex="-1"></a>                  <span class="at">groups =</span> qdata<span class="sc">$</span>merged)</span>
<span id="cb42-4"><a href="gm---size-class.html#cb42-4" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(pw.sz, </span>
<span id="cb42-5"><a href="gm---size-class.html#cb42-5" aria-hidden="true" tabindex="-1"></a>        <span class="at">confidence =</span> <span class="fl">0.95</span>, </span>
<span id="cb42-6"><a href="gm---size-class.html#cb42-6" aria-hidden="true" tabindex="-1"></a>        <span class="at">test.type =</span> <span class="st">&quot;dist&quot;</span>)</span></code></pre></div>
<pre><code>## 
## Pairwise comparisons
## 
## Groups: AN13_L AN13_S Millsap_L Millsap_S 
## 
## RRPP: 10000 permutations
## 
## LS means:
## Vectors hidden (use show.vectors = TRUE to view)
## 
## Pairwise distances between means, plus statistics
##                             d UCL (95%)          Z Pr &gt; d
## AN13_L:AN13_S        3.291613  20.45100 -0.7669429 0.7652
## AN13_L:Millsap_L     5.579910  18.24563 -0.1432140 0.5643
## AN13_L:Millsap_S    30.612284  17.25330  2.7917180 0.0001
## AN13_S:Millsap_L     8.871523  19.22403  0.3743559 0.3733
## AN13_S:Millsap_S    33.903897  18.05161  2.9388121 0.0001
## Millsap_L:Millsap_S 25.032374  15.83385  2.4948483 0.0011</code></pre>
</div>
<div id="trajectory-analysis" class="section level2 hasAnchor" number="3.5">
<h2><span class="header-section-number">3.5</span> Trajectory analysis<a href="gm---size-class.html#trajectory-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>A phenotypic trajectory analysis <span class="citation">(<a href="#ref-RN8344" role="doc-biblioref">Adams and Collyer 2007</a>, <a href="#ref-RN8550" role="doc-biblioref">2009</a>; <a href="#ref-RN8569" role="doc-biblioref">M. L. Collyer and Adams 2007</a>, <a href="#ref-RN8533" role="doc-biblioref">2013</a>; <a href="#ref-RN8446" role="doc-biblioref">Michael. L. Collyer, Sekora, and Adams 2015</a>)</span> was used to assess whether reduction/retouch trajectories differ between the sites.</p>
<div class="sourceCode" id="cb44"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb44-1"><a href="gm---size-class.html#cb44-1" aria-hidden="true" tabindex="-1"></a><span class="co"># trajectory analysis::shape</span></span>
<span id="cb44-2"><a href="gm---size-class.html#cb44-2" aria-hidden="true" tabindex="-1"></a>TA <span class="ot">&lt;-</span> <span class="fu">trajectory.analysis</span>(fit.sh,</span>
<span id="cb44-3"><a href="gm---size-class.html#cb44-3" aria-hidden="true" tabindex="-1"></a>                          <span class="at">groups =</span> qdata<span class="sc">$</span>site, </span>
<span id="cb44-4"><a href="gm---size-class.html#cb44-4" aria-hidden="true" tabindex="-1"></a>                          <span class="at">traj.pts =</span> qdata<span class="sc">$</span>size,</span>
<span id="cb44-5"><a href="gm---size-class.html#cb44-5" aria-hidden="true" tabindex="-1"></a>                          <span class="at">print.progress =</span> <span class="cn">FALSE</span>)</span>
<span id="cb44-6"><a href="gm---size-class.html#cb44-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb44-7"><a href="gm---size-class.html#cb44-7" aria-hidden="true" tabindex="-1"></a><span class="co"># magnitude difference</span></span>
<span id="cb44-8"><a href="gm---size-class.html#cb44-8" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(TA, </span>
<span id="cb44-9"><a href="gm---size-class.html#cb44-9" aria-hidden="true" tabindex="-1"></a>        <span class="at">attribute =</span> <span class="st">&quot;MD&quot;</span>)</span></code></pre></div>
<pre><code>## 
## Trajectory analysis
## 
## 10000 permutations.
## 
## Points projected onto trajectory PCs
## 
## Trajectories:
## Trajectories hidden (use show.trajectories = TRUE to view)
## 
## Observed path distances by group
## 
##        AN13     Millsap 
## 0.015387213 0.009509621 
## 
## Pairwise absolute differences in path distances, plus statistics
##                        d  UCL (95%)          Z Pr &gt; d
## AN13:Millsap 0.005877592 0.02094446 -0.1351729 0.5637</code></pre>
<div class="sourceCode" id="cb46"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb46-1"><a href="gm---size-class.html#cb46-1" aria-hidden="true" tabindex="-1"></a><span class="co"># plot</span></span>
<span id="cb46-2"><a href="gm---size-class.html#cb46-2" aria-hidden="true" tabindex="-1"></a>TP <span class="ot">&lt;-</span> <span class="fu">plot</span>(TA,</span>
<span id="cb46-3"><a href="gm---size-class.html#cb46-3" aria-hidden="true" tabindex="-1"></a>           <span class="at">pch =</span> <span class="fu">as.numeric</span>(qdata<span class="sc">$</span>site), </span>
<span id="cb46-4"><a href="gm---size-class.html#cb46-4" aria-hidden="true" tabindex="-1"></a>           <span class="at">bg =</span> <span class="fu">as.numeric</span>(qdata<span class="sc">$</span>size),</span>
<span id="cb46-5"><a href="gm---size-class.html#cb46-5" aria-hidden="true" tabindex="-1"></a>           <span class="at">cex =</span> <span class="fl">0.3</span>,</span>
<span id="cb46-6"><a href="gm---size-class.html#cb46-6" aria-hidden="true" tabindex="-1"></a>           <span class="at">col =</span> <span class="st">&quot;gray&quot;</span>)</span>
<span id="cb46-7"><a href="gm---size-class.html#cb46-7" aria-hidden="true" tabindex="-1"></a><span class="fu">add.trajectories</span>(TP, </span>
<span id="cb46-8"><a href="gm---size-class.html#cb46-8" aria-hidden="true" tabindex="-1"></a>                 <span class="at">traj.pch =</span> <span class="fu">c</span>(<span class="dv">15</span>, <span class="dv">17</span>),</span>
<span id="cb46-9"><a href="gm---size-class.html#cb46-9" aria-hidden="true" tabindex="-1"></a>                 <span class="at">traj.cex =</span> <span class="fl">0.5</span>)</span></code></pre></div>
<p><img src="supp_files/figure-html/shapetrajectory-1.png" width="100%" /></p>
</div>
<div id="morphological-disparity-1" class="section level2 hasAnchor" number="3.6">
<h2><span class="header-section-number">3.6</span> Morphological Disparity<a href="gm---size-class.html#morphological-disparity-1" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The trajectory analysis was followed by an analysis of morphological disparity <span class="citation">(<a href="#ref-RN11107" role="doc-biblioref">M. L. Collyer and Adams 2020</a>; <a href="#ref-RN7041" role="doc-biblioref">Foote 1993</a>; <a href="#ref-RN5694" role="doc-biblioref">Zelditch et al. 2004</a>)</span> to identify potential differences in morphological diversity between size classes.</p>
<div class="sourceCode" id="cb47"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb47-1"><a href="gm---size-class.html#cb47-1" aria-hidden="true" tabindex="-1"></a><span class="co"># morphological disparity ----</span></span>
<span id="cb47-2"><a href="gm---size-class.html#cb47-2" aria-hidden="true" tabindex="-1"></a><span class="co"># do any of the groups display greater shape variation among </span></span>
<span id="cb47-3"><a href="gm---size-class.html#cb47-3" aria-hidden="true" tabindex="-1"></a><span class="co"># individuals relative to the other group?</span></span>
<span id="cb47-4"><a href="gm---size-class.html#cb47-4" aria-hidden="true" tabindex="-1"></a><span class="fu">morphol.disparity</span>(shape <span class="sc">~</span> merged,</span>
<span id="cb47-5"><a href="gm---size-class.html#cb47-5" aria-hidden="true" tabindex="-1"></a>                  <span class="at">groups =</span> qdata<span class="sc">$</span>merged, </span>
<span id="cb47-6"><a href="gm---size-class.html#cb47-6" aria-hidden="true" tabindex="-1"></a>                  <span class="at">data =</span> gdf, </span>
<span id="cb47-7"><a href="gm---size-class.html#cb47-7" aria-hidden="true" tabindex="-1"></a>                  <span class="at">print.progress =</span> <span class="cn">FALSE</span>, </span>
<span id="cb47-8"><a href="gm---size-class.html#cb47-8" aria-hidden="true" tabindex="-1"></a>                  <span class="at">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
<pre><code>## 
## Call:
## morphol.disparity(f1 = shape ~ merged, groups = qdata$merged,      iter = 9999, data = gdf, print.progress = FALSE) 
## 
## 
## 
## Randomized Residual Permutation Procedure Used
## 10000 Permutations
## 
## Procrustes variances for defined groups
##      AN13_L      AN13_S   Millsap_L   Millsap_S 
## 0.001497570 0.000541002 0.002475642 0.001437581 
## 
## 
## Pairwise absolute differences between variances
##                 AN13_L       AN13_S    Millsap_L    Millsap_S
## AN13_L    0.000000e+00 0.0009565683 0.0009780712 5.998896e-05
## AN13_S    9.565683e-04 0.0000000000 0.0019346395 8.965793e-04
## Millsap_L 9.780712e-04 0.0019346395 0.0000000000 1.038060e-03
## Millsap_S 5.998896e-05 0.0008965793 0.0010380602 0.000000e+00
## 
## 
## P-Values
##           AN13_L AN13_S Millsap_L Millsap_S
## AN13_L    1.0000 0.3928    0.3306    0.9502
## AN13_S    0.3928 1.0000    0.0522    0.3641
## Millsap_L 0.3306 0.0522    1.0000    0.2190
## Millsap_S 0.9502 0.3641    0.2190    1.0000</code></pre>
<div class="sourceCode" id="cb49"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb49-1"><a href="gm---size-class.html#cb49-1" aria-hidden="true" tabindex="-1"></a><span class="co"># do any of the groups display greater size variation among </span></span>
<span id="cb49-2"><a href="gm---size-class.html#cb49-2" aria-hidden="true" tabindex="-1"></a><span class="co"># individuals relative to the other group?</span></span>
<span id="cb49-3"><a href="gm---size-class.html#cb49-3" aria-hidden="true" tabindex="-1"></a><span class="fu">morphol.disparity</span>(size <span class="sc">~</span> merged,</span>
<span id="cb49-4"><a href="gm---size-class.html#cb49-4" aria-hidden="true" tabindex="-1"></a>                  <span class="at">groups =</span> qdata<span class="sc">$</span>merged, </span>
<span id="cb49-5"><a href="gm---size-class.html#cb49-5" aria-hidden="true" tabindex="-1"></a>                  <span class="at">data =</span> gdf, </span>
<span id="cb49-6"><a href="gm---size-class.html#cb49-6" aria-hidden="true" tabindex="-1"></a>                  <span class="at">print.progress =</span> <span class="cn">FALSE</span>, </span>
<span id="cb49-7"><a href="gm---size-class.html#cb49-7" aria-hidden="true" tabindex="-1"></a>                  <span class="at">iter =</span> <span class="dv">9999</span>)</span></code></pre></div>
<pre><code>## 
## Call:
## morphol.disparity(f1 = size ~ merged, groups = qdata$merged,      iter = 9999, data = gdf, print.progress = FALSE) 
## 
## 
## 
## Randomized Residual Permutation Procedure Used
## 10000 Permutations
## 
## Procrustes variances for defined groups
##    AN13_L    AN13_S Millsap_L Millsap_S 
##  97.22506  26.58641 178.50404  47.09519 
## 
## 
## Pairwise absolute differences between variances
##             AN13_L    AN13_S Millsap_L Millsap_S
## AN13_L     0.00000  70.63865  81.27898  50.12987
## AN13_S    70.63865   0.00000 151.91763  20.50878
## Millsap_L 81.27898 151.91763   0.00000 131.40885
## Millsap_S 50.12987  20.50878 131.40885   0.00000
## 
## 
## P-Values
##           AN13_L AN13_S Millsap_L Millsap_S
## AN13_L    1.0000 0.5090    0.4144    0.6284
## AN13_S    0.5090 1.0000    0.1393    0.8406
## Millsap_L 0.4144 0.1393    1.0000    0.1167
## Millsap_S 0.6284 0.8406    0.1167    1.0000</code></pre>

</div>
</div>
<h3>References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-RN8344" class="csl-entry">
Adams, Dean C., and Michael L. Collyer. 2007. <span>“Analysis of Character Divergence Along Environmental Gradients and Other Covariates.”</span> Journal Article. <em>Evolution</em> 61 (3): 510–15. <a href="https://doi.org/10.1111/j.1558-5646.2007.00063.x">https://doi.org/10.1111/j.1558-5646.2007.00063.x</a>.
</div>
<div id="ref-RN8550" class="csl-entry">
———. 2009. <span>“A General Framework for the Analysis of Phenotypic Trajectories in Evolutionary Studies.”</span> Journal Article. <em>Evolution</em> 63 (5): 1143–54. <a href="https://doi.org/10.1111/j.1558-5646.2009.00649.x">https://doi.org/10.1111/j.1558-5646.2009.00649.x</a>.
</div>
<div id="ref-RN1655" class="csl-entry">
———. 2015. <span>“<span class="nocase">Permutation Tests for Phylogenetic Comparative Analyses of High-Dimensional Shape Data: What you Shuffle Matters</span>.”</span> <em>Evolution</em> 69 (3): 823–29. <a href="https://doi.org/10.1111/evo.12596">https://doi.org/10.1111/evo.12596</a>.
</div>
<div id="ref-RN11530" class="csl-entry">
Adams, Dean C., Michael L. Collyer, Antigoni Kaliontzopoulou, and Emma Sherratt. 2017. <span>“<span class="nocase">Package geomorph: Geometric Morphometric Analyses of Landmark Data. R package version 3.0.5</span>.”</span> <a href="http://geomorphr.github.io/geomorph/">http://geomorphr.github.io/geomorph/</a>.
</div>
<div id="ref-RN1774" class="csl-entry">
Adams, Dean C., and Erik Otarola-Castillo. 2013. <span>“<span class="nocase">geomorph: An R Package for the Collection and Analysis of Geometric Morphometric Shape Data</span>.”</span> <em>Methods in Ecology and Evolution</em> 4 (4): 393–99. <a href="https://doi.org/10.1111/2041-210x.12035">https://doi.org/10.1111/2041-210x.12035</a>.
</div>
<div id="ref-RN1719" class="csl-entry">
Anderson, M. J., and C. J. F. Ter Braak. 2003. <span>“<span class="nocase">Permutation Tests for Multi-Factoral Analysis of Variance</span>.”</span> <em>Journal of Statistical Computation and Simulation</em> 73 (2): 85–113. <a href="https://doi.org/10.1080=0094965021000015558">https://doi.org/10.1080=0094965021000015558</a>.
</div>
<div id="ref-RN9565" class="csl-entry">
Baken, Erica K., Michael L. Collyer, Antigoni Kaliontzopoulou, and Dean C. Adams. 2021. <span>“<span class="nocase">geomorph v4.0 and gmShiny: Enhanced analytics and a new graphical interface for a comprehensive morphometric experience</span>.”</span> Journal Article. <em>Methods in Ecology and Evolution</em>. <a href="https://doi.org/10.1111/2041-210x.13723">https://doi.org/10.1111/2041-210x.13723</a>.
</div>
<div id="ref-RN8569" class="csl-entry">
Collyer, Michael L., and Dean C. Adams. 2007. <span>“Analysis of Two-State Multivariate Phenotypic Change in Ecological Studies.”</span> Journal Article. <em>Ecology</em> 88 (3): 683–92. <a href="https://doi.org/10.1890/06-0727">https://doi.org/10.1890/06-0727</a>.
</div>
<div id="ref-RN8533" class="csl-entry">
———. 2013. <span>“Phenotypic Trajectory Analysis: Comparison of Shape Change Patterns in Evolution and Ecology.”</span> Journal Article. <em>Hystrix</em> 24 (1): 75–83. <a href="https://doi.org/doi:10.4404/hystrix-24.1-6298">https://doi.org/doi:10.4404/hystrix-24.1-6298</a>.
</div>
<div id="ref-RN11775" class="csl-entry">
———. 2018. <span>“<span class="nocase">RRPP: An R Package for Fitting Linear Models to High-Dimensional Data using Residual Randomization</span>.”</span> <em>Methods in Ecology and Evolution</em> 9 (7): 1772–79. https://doi.org/<a href="https://doi.org/10.1111/2041-210X.13029">https://doi.org/10.1111/2041-210X.13029</a>.
</div>
<div id="ref-RN11107" class="csl-entry">
———. 2020. <span>“Phylogenetically Aligned Component Analysis.”</span> Journal Article. <em>Methods in Ecology and Evolution</em> 12 (2): 359–72. <a href="https://doi.org/10.1111/2041-210x.13515">https://doi.org/10.1111/2041-210x.13515</a>.
</div>
<div id="ref-RN1756" class="csl-entry">
Collyer, Michael L., D. J. Sekora, and Dean C. Adams. 2015. <span>“<span class="nocase">A Method for Analysis of Phenotypic Change for Phenotypes Described by High-Dimensional Data</span>.”</span> <em>Heredity</em> 115 (4): 357–65. <a href="https://doi.org/10.1038/hdy.2014.75">https://doi.org/10.1038/hdy.2014.75</a>.
</div>
<div id="ref-RN8446" class="csl-entry">
Collyer, Michael. L., David. J. Sekora, and Dean. C. Adams. 2015. <span>“A Method for Analysis of Phenotypic Change for Phenotypes Described by High-Dimensional Data.”</span> Journal Article. <em>Heredity (Edinb)</em> 115 (4): 357–65. <a href="https://doi.org/10.1038/hdy.2014.75">https://doi.org/10.1038/hdy.2014.75</a>.
</div>
<div id="ref-RN7041" class="csl-entry">
Foote, M. 1993. <span>“Contributions of Individual Taxa to Overall Morphological Disparity.”</span> Journal Article. <em>Paleobiology</em> 19 (4): 403–19.
</div>
<div id="ref-RN1749" class="csl-entry">
Goodall, Colin. 1991. <span>“<span class="nocase">Procrustes Methods in the Statistical Analysis of Shape</span>.”</span> <em>Journal of the Royal Statistical Society. Series B (Methodological)</em> 53 (2): 285–339.
</div>
<div id="ref-RN11564" class="csl-entry">
Gower, J. C. 1975. <span>“<span>Generalized Procrustes Analysis</span>.”</span> <em>Psychometrika</em> 40 (1): 33–51. https://doi.org/<a href="https://doi.org/10.1007/BF02291478">https://doi.org/10.1007/BF02291478</a>.
</div>
<div id="ref-RN1746" class="csl-entry">
Jolliffe, Ian T. 2002. <em>Principal Component Analysis</em>. New York: Springer.
</div>
<div id="ref-RN11622" class="csl-entry">
Kendall, David G. 1981. <span>“<span class="nocase">The Statistics of Shape</span>.”</span> In <em>Interpreting Multivariate Data</em>, edited by V. Barnett, 75–80. New York: Wiley.
</div>
<div id="ref-RN11623" class="csl-entry">
———. 1984. <span>“<span class="nocase">Shape Manifolds, Procrustean Metrics, and Complex Projective Spaces</span>.”</span> <em>Bulletin of the London Mathematical Society</em> 16 (2): 81–121. <a href="https://doi.org/10.1112/blms/16.2.81">https://doi.org/10.1112/blms/16.2.81</a>.
</div>
<div id="ref-RN1731" class="csl-entry">
Klingenberg, Christian Peter. 2013. <span>“<span class="nocase">Visualizations in Geometric Morphometrics: How to Read and How to Make Graphs Showing Shape Changes</span>.”</span> <em>Hystrix</em> 24 (1): 15–24. https://doi.org/<a href="http://dx.doi.org/10.4404/hystrix-24.1-7691">http://dx.doi.org/10.4404/hystrix-24.1-7691</a>.
</div>
<div id="ref-R" class="csl-entry">
R Core Development Team,. 2022. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="http://www.R-project.org/">http://www.R-project.org/</a>.
</div>
<div id="ref-RN1646" class="csl-entry">
Rohlf, F. James. 1999. <span>“<span class="nocase">Shape Statistics: Procrustes Superimpositions and Tangent Spaces</span>.”</span> <em>Journal of Classification</em> 16 (2): 197–223. <a href="https://doi.org/10.1007/s003579900054">https://doi.org/10.1007/s003579900054</a>.
</div>
<div id="ref-RN478" class="csl-entry">
Rohlf, F. James, and Dennis Slice. 1990. <span>“<span class="nocase">Extensions of the Procrustes Method for the Optimal Superimposition of Landmarks</span>.”</span> <em>Systematic Zoology</em> 39 (1): 40–59. <a href="https://doi.org/10.2307/2992207">https://doi.org/10.2307/2992207</a>.
</div>
<div id="ref-RN9551" class="csl-entry">
———. 2021b. <span>“<span class="nocase">Shape as a function of time + raw material + burial context? An exploratory analysis of Perdiz arrow points from the ancestral Caddo area of the American Southeast</span>.”</span> Journal Article. <em>Journal of Archaeological Science: Reports</em> 37: 102916. <a href="https://doi.org/10.1016/j.jasrep.2021.102916">https://doi.org/10.1016/j.jasrep.2021.102916</a>.
</div>
<div id="ref-RN11801" class="csl-entry">
Selden Jr., Robert Z. 2018a. <span>“A Preliminary Study of Smithport Plain Bottle Morphology in the Southern Caddo Area.”</span> <em>Bulletin of the Texas Archeological Society</em> 89: 63–89.
</div>
<div id="ref-RN11782" class="csl-entry">
———. 2018b. <span>“<span class="nocase">Ceramic Morphological Organisation in the Southern Caddo Area: Quiddity of Shape for Hickory Engraved Bottles</span>.”</span> <em>Journal of Archaeological Science: Reports</em> 21: 884–96. <a href="https://doi.org/10.1016/j.jasrep.2018.08.045">https://doi.org/10.1016/j.jasrep.2018.08.045</a>.
</div>
<div id="ref-RN11716" class="csl-entry">
———. 2019. <span>“<span class="nocase">Ceramic Morphological Organisation in the Southern Caddo Area: The Clarence H. Webb Collections</span>.”</span> <em>Journal of Cultural Heritage</em> 35: 41–55. https://doi.org/<a href="https://doi.org/10.1016/j.culher.2018.07.002">https://doi.org/10.1016/j.culher.2018.07.002</a>.
</div>
<div id="ref-RN8312" class="csl-entry">
———. 2021. <span>“Louisiana Limitrophe: An Iterative Morphological Exegesis of Caddo Bottle and Biface Production.”</span> Book Section. In <em>Ancestral Caddo Ceramic Traditions</em>, edited by Duncan P. McKinnon, Jeffrey S. Girard, and Timothy K. Perttula, 258–76. Baton Rouge: LSU Press.
</div>
<div id="ref-RN21001" class="csl-entry">
Selden Jr., Robert Z., John E. Dockall, and Morgane Dubied. 2020. <span>“A Quantitative Assessment of Intraspecific Morphological Variation in Gahagan Bifaces from the Southern Caddo Area and Central Texas.”</span> <em>Southeastern Archaeology</em> 39 (2): 125–45. <a href="https://doi.org/10.1080/0734578x.2020.1744416">https://doi.org/10.1080/0734578x.2020.1744416</a>.
</div>
<div id="ref-RN11783" class="csl-entry">
Selden Jr., Robert Z., John E. Dockall, and Harry J. Shafer. 2018. <span>“<span class="nocase">Lithic Morphological Organisation: Gahagan Bifaces from the Southern Caddo Area</span>.”</span> <em>Digital Applications in Archaeology and Cultural Heritage</em> 10: e00080. <a href="https://doi.org/10.1016/j.daach.2018.e00080">https://doi.org/10.1016/j.daach.2018.e00080</a>.
</div>
<div id="ref-RN11264" class="csl-entry">
Selden, Robert Z. 2022. <span>“Morphologically Similar, but Regionally Distinct: Perdiz Arrow Points from Caddo Burial Contexts in the American Southeast.”</span> Journal Article. <em>Lithic Technology</em> (in press). <a href="https://doi.org/10.1080/01977261.2022.2095492">https://doi.org/10.1080/01977261.2022.2095492</a>.
</div>
<div id="ref-RN479" class="csl-entry">
Sherratt, Emma, David J. Gower, Christian Peter Klingenberg, and Mark Wilkinson. 2014. <span>“<span class="nocase">Evolution of Cranial Shape in Caecilians (Amphibia: Gymnophiona)</span>.”</span> <em>Evolutionary Biology</em> 41 (4): 528–45. https://doi.org/<a href="https://doi.org/10.1007/s11692-014-9287-2">https://doi.org/10.1007/s11692-014-9287-2</a>.
</div>
<div id="ref-RN11563" class="csl-entry">
Slice, Dennis E. 2001. <span>“<span class="nocase">Landmark Coordinates Aligned by Procrustes Analysis Do Not Lie in Kendall’s Shape Space</span>.”</span> <em>Systematic Biology</em> 50 (1): 141–49. <a href="https://doi.org/10.1080/10635150119110">https://doi.org/10.1080/10635150119110</a>.
</div>
<div id="ref-RN5694" class="csl-entry">
Zelditch, Miriam Leah, Donald L. Swiderski, H. David Sheets, and William L. Fink. 2004. <em>Geometric Morphometrics for Biologists : A Primer</em>. Book. Burlington: Elsevier Science. <a href="http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308">http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="gm---siteraw-material.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "night",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["supp.pdf", "supp.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
