<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Caddo retouch results in divergent morphology for elliptical bifaces from sites in the American Southeast</title>
  <meta name="description" content="Pilot study of biface morphology." />
  <meta name="generator" content="bookdown 0.27 and GitBook 2.6.7" />

  <meta property="og:title" content="Caddo retouch results in divergent morphology for elliptical bifaces from sites in the American Southeast" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Pilot study of biface morphology." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Caddo retouch results in divergent morphology for elliptical bifaces from sites in the American Southeast" />
  
  <meta name="twitter:description" content="Pilot study of biface morphology." />
  

<meta name="author" content="Robert Z. Selden, Jr. and John E. Dockall" />


<meta name="date" content="2022-08-05" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  

<link rel="next" href="define-size-classes.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="custom.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">jowell.1</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a>
<ul>
<li class="chapter" data-level="0.1" data-path="index.html"><a href="index.html#analysis-overview"><i class="fa fa-check"></i><b>0.1</b> Analysis overview</a>
<ul>
<li class="chapter" data-level="0.1.1" data-path="index.html"><a href="index.html#generalised-procrustes-analysis"><i class="fa fa-check"></i><b>0.1.1</b> Generalised Procrustes Analysis</a></li>
<li class="chapter" data-level="0.1.2" data-path="index.html"><a href="index.html#principal-components-analysis"><i class="fa fa-check"></i><b>0.1.2</b> Principal Components Analysis</a></li>
<li class="chapter" data-level="0.1.3" data-path="index.html"><a href="index.html#procrustes-anova"><i class="fa fa-check"></i><b>0.1.3</b> Procrustes ANOVA</a></li>
<li class="chapter" data-level="0.1.4" data-path="index.html"><a href="index.html#morhpological-disparity"><i class="fa fa-check"></i><b>0.1.4</b> Morhpological disparity</a></li>
</ul></li>
<li class="chapter" data-level="0.2" data-path="index.html"><a href="index.html#primary-findings"><i class="fa fa-check"></i><b>0.2</b> Primary findings</a>
<ul>
<li class="chapter" data-level="0.2.1" data-path="index.html"><a href="index.html#summary"><i class="fa fa-check"></i><b>0.2.1</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="0.3" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i><b>0.3</b> Acknowledgments</a></li>
<li class="chapter" data-level="0.4" data-path="index.html"><a href="index.html#funding"><i class="fa fa-check"></i><b>0.4</b> Funding</a></li>
<li class="chapter" data-level="0.5" data-path="index.html"><a href="index.html#data-management"><i class="fa fa-check"></i><b>0.5</b> Data management</a></li>
<li class="chapter" data-level="0.6" data-path="index.html"><a href="index.html#colophon"><i class="fa fa-check"></i><b>0.6</b> Colophon</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="define-size-classes.html"><a href="define-size-classes.html"><i class="fa fa-check"></i><b>1</b> Define size classes</a>
<ul>
<li class="chapter" data-level="1.1" data-path="define-size-classes.html"><a href="define-size-classes.html#load-packages-data"><i class="fa fa-check"></i><b>1.1</b> Load packages + data</a></li>
<li class="chapter" data-level="1.2" data-path="define-size-classes.html"><a href="define-size-classes.html#generalised-procrustes-analysis-1"><i class="fa fa-check"></i><b>1.2</b> Generalised Procrustes Analysis</a></li>
<li class="chapter" data-level="1.3" data-path="define-size-classes.html"><a href="define-size-classes.html#assign-size-class"><i class="fa fa-check"></i><b>1.3</b> Assign size class</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html"><i class="fa fa-check"></i><b>2</b> GM - Site/Raw Material</a>
<ul>
<li class="chapter" data-level="2.1" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html#load-packages-data-1"><i class="fa fa-check"></i><b>2.1</b> Load packages + data</a></li>
<li class="chapter" data-level="2.2" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html#generalised-procrustes-analysis-2"><i class="fa fa-check"></i><b>2.2</b> Generalised Procrustes Analysis</a></li>
<li class="chapter" data-level="2.3" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html#principal-components-analysis-1"><i class="fa fa-check"></i><b>2.3</b> Principal Components Analysis</a></li>
<li class="chapter" data-level="2.4" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html#procrustes-anova-1"><i class="fa fa-check"></i><b>2.4</b> Procrustes ANOVA</a></li>
<li class="chapter" data-level="2.5" data-path="gm---siteraw-material.html"><a href="gm---siteraw-material.html#morphological-disparity"><i class="fa fa-check"></i><b>2.5</b> Morphological Disparity</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="gm---size-class.html"><a href="gm---size-class.html"><i class="fa fa-check"></i><b>3</b> GM - Size Class</a>
<ul>
<li class="chapter" data-level="3.1" data-path="gm---size-class.html"><a href="gm---size-class.html#load-packages-data-2"><i class="fa fa-check"></i><b>3.1</b> Load packages + data</a></li>
<li class="chapter" data-level="3.2" data-path="gm---size-class.html"><a href="gm---size-class.html#generalised-procrustes-analysis-3"><i class="fa fa-check"></i><b>3.2</b> Generalised Procrustes Analysis</a></li>
<li class="chapter" data-level="3.3" data-path="gm---size-class.html"><a href="gm---size-class.html#principal-components-analysis-2"><i class="fa fa-check"></i><b>3.3</b> Principal Components Analysis</a></li>
<li class="chapter" data-level="3.4" data-path="gm---size-class.html"><a href="gm---size-class.html#procrustes-anova-2"><i class="fa fa-check"></i><b>3.4</b> Procrustes ANOVA</a></li>
<li class="chapter" data-level="3.5" data-path="gm---size-class.html"><a href="gm---size-class.html#morphological-disparity-1"><i class="fa fa-check"></i><b>3.5</b> Morphological Disparity</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Caddo retouch results in divergent morphology for elliptical bifaces from sites in the American Southeast</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">Caddo retouch results in divergent morphology for elliptical bifaces from sites in the American Southeast</h1>
<p class="author"><em>Robert Z. Selden, Jr.<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a> and John E. Dockall<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a></em></p>
<p class="date"><em>2022-08-05</em></p>
</div>
<div id="preface" class="section level1 unnumbered hasAnchor">
<h1>Preface<a href="index.html#preface" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<blockquote>
<p>Reproducibility—the ability to recompute results—and replicability—the chances other experimenters will achieve a consistent result—are two foundational characteristics of successful scientific research <span class="citation">(<a href="#ref-RN20918" role="doc-biblioref">Leek and Peng 2015</a>)</span>.</p>
</blockquote>
<p>Elliptical bifaces from the Millsap Cache and Jowell Farm (41AN13) sites comprise two of the largest samples of this ill-understood lithic tool from the ancestral Caddo area. Similar to Gahagan bifaces, the bifaces from Jowell Farm are assumed to have been manufactured using Edwards chert—due to their size, the colours that they fluoresce beneath low and high frequency UV light, and the paucity of similarly large locally-available raw material sources—and acquired by the Caddo through trade with central Texas groups. Bifaces from the Millsap Cache are thought to have been manufactured from Kay County flint, providing for an empirical assessment of whether elliptical biface morphology differs as a function of raw material and/or biogeographical range. The sample from each site was subsequently divided into two size classes; one reflective of initial production/manufacture (larger), and the other associated with resharpening and/or retouch (smaller). Size classes were used to assess whether modifications by Caddo knappers may have yielded a similar—convergent—knife shape at the smaller size class, and to test the hypothesis that greater morphological variation would be apparent in that size class due to wear associated with regular use, paired with variable idiosyncracies related to resharpening and/or retouch. Results demonstrate that elliptical biface shape does not differ by raw material, but size does. This finding is suggestive of a shared mental template, but a size differential that is potentially driven by raw material and/or biogeographical differences. Additionally, elliptical biface size associated with the large size class does not differ between sites/raw materials; however, the small size class does. This finding aids in advancing the argument that elliptical biface morphology diverges through the resharpening trajectory, where bifaces from each site/raw material enter their uselife at a similar shape and size, but diverge significantly in size—but not shape—throughout that uselife. In following with our expectations, greater shape diversity occurred in the small size class, where the bifaces from Jowell Farm were found to be more diverse than those from Millsap Cache. Discrete biogeographical ranges and raw material differences are posited as the drivers of these extant morphological differences found in elliptical bifaces from the ancestral Caddo area.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="index.html#cb1-1" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">&#39;./images/map.png&#39;</span>)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:figmap"></span>
<img src="images/map.png" alt="Geological map illustrating the location of the Millsap Cache and Jowell Farm, the level three ecoregions of the Arkansas Valley (blue), Ouachita Mountains (yellow), South Central Plains (orange), and ancestral Caddo area boundary (white) in the American Southeast." width="1650" />
<p class="caption">
Figure 0.1: Geological map illustrating the location of the Millsap Cache and Jowell Farm, the level three ecoregions of the Arkansas Valley (blue), Ouachita Mountains (yellow), South Central Plains (orange), and ancestral Caddo area boundary (white) in the American Southeast.
</p>
</div>
<div id="analysis-overview" class="section level2 hasAnchor" number="0.1">
<h2><span class="header-section-number">0.1</span> Analysis overview<a href="index.html#analysis-overview" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Prior to landmarking, elliptical bifaces were oriented with the most heavily retouched edge at top right (Figure <a href="index.html#fig:figlm">0.2</a>). The landmarking protocol uses three landmarks; two horizontal tangents (top/bottom), and one placed at the furthest extent of the area that includes the heaviest amount of retouch; all were applied using the <code>StereoMorph</code> package <span class="citation">(<a href="#ref-RN8973" role="doc-biblioref">Olsen and Westneat 2015</a>)</span>. Some bifaces include multiple areas of heavy retouch at the top and bottom of the same lateral edge (Figure <a href="index.html#fig:figlm">0.2</a>: b, c, d, and e), while others (Figure <a href="index.html#fig:figlm">0.2</a>: f) include one retouched edge at top right and another at bottom left.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="index.html#cb2-1" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">&#39;./images/ellipticalbifaces.jpg&#39;</span>)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:figlm"></span>
<img src="images/ellipticalbifaces.jpg" alt="Selected elliptical bifaces from the Jowell Farm site, highlighting the most heavily retouched area identified using a modified version of the approach developed and advanced by Miller [-@RN11459], Smallwood [-@RN8310], and Miller and Smallwood [-@RN11460]." width="2100" />
<p class="caption">
Figure 0.2: Selected elliptical bifaces from the Jowell Farm site, highlighting the most heavily retouched area identified using a modified version of the approach developed and advanced by Miller <span class="citation">(<a href="#ref-RN11459" role="doc-biblioref">2007</a>)</span>, Smallwood <span class="citation">(<a href="#ref-RN8310" role="doc-biblioref">2010</a>)</span>, and Miller and Smallwood <span class="citation">(<a href="#ref-RN11460" role="doc-biblioref">2012</a>)</span>.
</p>
</div>
<p>To identify which edge was most heavily retouched, we employed a modified approach to the flaking index initially developed by Miller <span class="citation">(<a href="#ref-RN11459" role="doc-biblioref">2007</a>)</span>, and later advanced and refined by Smallwood <span class="citation">(<a href="#ref-RN8310" role="doc-biblioref">2010</a>)</span>, then Miller and Smallwood <span class="citation">(<a href="#ref-RN11460" role="doc-biblioref">2012</a>)</span>. The modified approach uses counts of flake scars from each edge in the two most heavily worked areas, paired with a measure of edge length (not linear/includes curvature). The number of flake scars was subsequently divided by the length of the worked edge, with the heaviest worked area identified by the greater value.</p>
<p>The pattern that emerged suggested that the longest reworked edge on the elliptical bifaces was always more heavily retouched. This finding was used to incorporate the images of elliptical bifaces from the Millsap Cache <span class="citation">(<a href="#ref-RN11461" role="doc-biblioref">Millsap and Dickson 1968</a>)</span>, where the longest reworked edges were oriented to the top right prior to landmarking.</p>
<div id="generalised-procrustes-analysis" class="section level3 hasAnchor" number="0.1.1">
<h3><span class="header-section-number">0.1.1</span> Generalised Procrustes Analysis<a href="index.html#generalised-procrustes-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Landmark data were subsequently aligned to a global coordinate system <span class="citation">(<a href="#ref-RN11622" role="doc-biblioref">Kendall 1981</a>, <a href="#ref-RN11623" role="doc-biblioref">1984</a>; <a href="#ref-RN11563" role="doc-biblioref">Slice 2001</a>)</span>, achieved through generalized Procrustes superimposition <span class="citation">(<a href="#ref-RN478" role="doc-biblioref">Rohlf and Slice 1990</a>)</span> performed in R 4.2.1 <span class="citation">(<a href="#ref-R" role="doc-biblioref">R Core Development Team, 2022</a>)</span> using the <code>geomorph</code> library v. 4.0.4 <span class="citation">(<a href="#ref-RN11530" role="doc-biblioref">Adams et al. 2017</a>; <a href="#ref-RN1774" role="doc-biblioref">Adams and Otarola-Castillo 2013</a>; <a href="#ref-RN9565" role="doc-biblioref">Baken et al. 2021</a>)</span>. Procrustes superimposition translates, scales, and rotates the coordinate data to allow for comparisons among objects <span class="citation">(<a href="#ref-RN11564" role="doc-biblioref">Gower 1975</a>; <a href="#ref-RN478" role="doc-biblioref">Rohlf and Slice 1990</a>)</span>. The <code>geomorph</code> package uses a partial Procrustes superimposition that projects the aligned specimens into tangent space subsequent to alignment in preparation for the use of multivariate methods that assume linear space <span class="citation">(<a href="#ref-RN1646" role="doc-biblioref">Rohlf 1999</a>; <a href="#ref-RN11563" role="doc-biblioref">Slice 2001</a>)</span>.</p>
</div>
<div id="principal-components-analysis" class="section level3 hasAnchor" number="0.1.2">
<h3><span class="header-section-number">0.1.2</span> Principal Components Analysis<a href="index.html#principal-components-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Principal components analysis <span class="citation">(<a href="#ref-RN1746" role="doc-biblioref">Jolliffe 2002</a>)</span> was used to visualise shape variation among the elliptical bifaces. The shape changes described by each principal axis are commonly visualized using thin-plate spline warping of a reference image or 3D mesh <span class="citation">(<a href="#ref-RN1731" role="doc-biblioref">Klingenberg 2013</a>; <a href="#ref-RN479" role="doc-biblioref">Sherratt et al. 2014</a>)</span>.</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="index.html#cb3-1" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">include_graphics</span>(<span class="st">&#39;./images/pca.png&#39;</span>)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:figjowellpca"></span>
<img src="images/pca.png" alt="PCA of shape variables for elogated ovate bifaces." width="2100" />
<p class="caption">
Figure 0.3: PCA of shape variables for elogated ovate bifaces.
</p>
</div>
</div>
<div id="procrustes-anova" class="section level3 hasAnchor" number="0.1.3">
<h3><span class="header-section-number">0.1.3</span> Procrustes ANOVA<a href="index.html#procrustes-anova" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>To assess whether shape and size differed by <code>raw material</code> and <code>size class</code>, Procrustes ANOVAs <span class="citation">(<a href="#ref-RN1749" role="doc-biblioref">Goodall 1991</a>)</span> were run that enlist effect-sizes (zscores) computed as standard deviates of the generated sampling distributions <span class="citation">(<a href="#ref-RN1756" role="doc-biblioref">Collyer, Sekora, and Adams 2015</a>)</span>. A residual randomization permutation procedure (RRPP; n = 10,000 permutations) was used for all Procrustes ANOVAs <span class="citation">(<a href="#ref-RN1655" role="doc-biblioref">Adams and Collyer 2015</a>; <a href="#ref-RN11775" role="doc-biblioref">Collyer and Adams 2018</a>)</span>, which has higher statistical power and a greater ability to identify patterns in the data should they be present <span class="citation">(<a href="#ref-RN1719" role="doc-biblioref">Anderson and Ter Braak 2003</a>)</span>.</p>
</div>
<div id="morhpological-disparity" class="section level3 hasAnchor" number="0.1.4">
<h3><span class="header-section-number">0.1.4</span> Morhpological disparity<a href="index.html#morhpological-disparity" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>An analysis of morphological disparity was used <span class="citation">(<a href="#ref-RN11107" role="doc-biblioref">Collyer and Adams 2020</a>; <a href="#ref-RN7041" role="doc-biblioref">Foote 1993</a>; <a href="#ref-RN5694" role="doc-biblioref">Zelditch et al. 2004</a>)</span> to identify potential differences in morphological diversity between size classes.</p>
</div>
</div>
<div id="primary-findings" class="section level2 hasAnchor" number="0.2">
<h2><span class="header-section-number">0.2</span> Primary findings<a href="index.html#primary-findings" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ul>
<li>Bifaces <em>do not differ</em> in shape, but <strong>differ significantly</strong> in size by raw material/biogeography.</li>
<li>Bifaces in the <em>large size class</em> recovered from the Millsap Cache and Jowell Farm do not differ in shape or size. This suggests consistent morphology in bifaces from the <em>large size class</em>, in support of production-based standardization.</li>
<li>Bifaces in the <em>small size class</em> recovered from the Millsap Cache and Jowell Farm <strong>differ significantly</strong> in size. Thus, while shape remains consistent, size differs, suggesting—at minimum—locally divergent approaches to retouch and/or resharpening.</li>
<li>Morphological disparity in the shape of the <em>small size class</em> is <strong>significantly greater</strong> at the Jowell Farm site, providing evidence for a more standardized approach to resharpening and/or retouch at the Millsap Cache site. That finding also supports the hypothesis that greater diversity in shape is associated with retouch and/or resharpening, rather than production/manufacture.</li>
</ul>
<div id="summary" class="section level3 hasAnchor" number="0.2.1">
<h3><span class="header-section-number">0.2.1</span> Summary<a href="index.html#summary" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The shape of elliptical bifaces does not differ significantly in either size class; however, size does. Results of the Procrustes ANOVA support the hypothesis for divergent morphology. The analysis of morphological disparity yielded support for the interpretation that greater morphological diversity is associated with resharpening and/or retouch, while providing additional support for morphological consistency in production/manufacture.</p>
</div>
</div>
<div id="acknowledgments" class="section level2 hasAnchor" number="0.3">
<h2><span class="header-section-number">0.3</span> Acknowledgments<a href="index.html#acknowledgments" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Our thanks to the Caddo Nation of Oklahoma, the Caddo Nation Tribal Council, Tribal Chairman, and Tribal Historic Preservation Office for permission and access to NAGPRA and previously repatriated collections. Our gratitude is also extended to Marybeth Tomka and Lauren Bussiere at the Texas Archeological Research Laboratory for their assistance with access to the bifaces and the associated records, and to Scott Hammerstedt and Debra K. Green at the Oklahoma Archeological Survey for their assistance with records requests. Thanks also to John Harman for access to the DStretch plugin for ImageJ that was useful in the analysis of flake scars, and to Harry J. Shafer, Hiram F. (Pete) Gregory, Christian S. Hoggard, and David K. Thulman for their comments and constructive criticisms on the ongoing analyses of Caddo biface morphology, and to Emma Sherratt, Kersten Bergstrom, Dean C. Adams, and Michael L. Collyer for their constructive criticisms, general comments, and suggestions throughout the development of this research program.</p>
</div>
<div id="funding" class="section level2 hasAnchor" number="0.4">
<h2><span class="header-section-number">0.4</span> Funding<a href="index.html#funding" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Components of this analytical work flow were developed and funded by a Preservation Technology and Training grant (P14AP00138) to RZS from the National Center for Preservation Technology and Training (NCPTT), and additional grants to RZS from the Caddo Nation of Oklahoma, National Forests and Grasslands in Texas (15-PA-11081300-033) and the United States Forest Service (20-PA-11081300-074). Funding to analyse the bifaces from the Millsap Cache and Jowell Farm was provided by the Heritage Research Center at Stephen F. Austin State University.</p>
</div>
<div id="data-management" class="section level2 hasAnchor" number="0.5">
<h2><span class="header-section-number">0.5</span> Data management<a href="index.html#data-management" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The analysis code associated with this project can be accessed through this document, is available in the GitHub repository, and digitally curated on the Open Science Framework. The reproducible nature of this enterprise provides a means for others to critically assess and evaluate the various analytical components <span class="citation">(<a href="#ref-RN20915" role="doc-biblioref">Gray and Marwick 2019</a>; <a href="#ref-RN20916" role="doc-biblioref">Peng 2011</a>; <a href="#ref-RN20917" role="doc-biblioref">Gandrud 2014</a>)</span>, which is a necessary requirement for the production of reliable knowledge.</p>
<p>Reproducibility projects in <a href="https://osf.io/ezcuj/">psychology</a> and <a href="https://www.cos.io/rpcb">cancer biology</a> are impacting current research practices across all domains. Examples of reproducible research are becoming more abundant in archaeology <span class="citation">(<a href="#ref-RN20804" role="doc-biblioref">Marwick 2016</a>; <a href="#ref-RN21009" role="doc-biblioref">Ivanovaite et al. 2020</a>; <a href="#ref-RN11783" role="doc-biblioref">Selden Jr., Dockall, and Shafer 2018</a>; <a href="#ref-RN21001" role="doc-biblioref">Selden Jr., Dockall, and Dubied 2020</a>; <a href="#ref-RN9364" role="doc-biblioref">Selden Jr et al. 2021a</a>)</span>, and the next generation of archaeologists are learning those tools and methods needed to reproduce and/or replicate research results <span class="citation">(<a href="#ref-RN21007" role="doc-biblioref">Marwick et al. 2019</a>)</span>. Reproducible and replicable research work flows are often employed at the highest levels of humanities-based inquiries to mitigate concern or doubt regarding proper execution, and is of particular import should the results have—explicitly or implicitly—a major impact on scientific progress <span class="citation">(<a href="#ref-RN21008" role="doc-biblioref">Peels and Bouter 2018</a>)</span>.</p>
</div>
<div id="colophon" class="section level2 hasAnchor" number="0.6">
<h2><span class="header-section-number">0.6</span> Colophon<a href="index.html#colophon" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>This version of the analysis was generated on 2022-08-05 08:49:04 using the following computational environment and dependencies:</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="index.html#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="co"># what R packages and versions were used?</span></span>
<span id="cb4-2"><a href="index.html#cb4-2" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> (<span class="st">&quot;devtools&quot;</span> <span class="sc">%in%</span> <span class="fu">installed.packages</span>()) devtools<span class="sc">::</span><span class="fu">session_info</span>()</span></code></pre></div>
<pre><code>## ─ Session info ────────────────────────────────────────────────────────────────────────────────────
##  setting  value
##  version  R version 4.2.1 (2022-06-23 ucrt)
##  os       Windows 10 x64 (build 19044)
##  system   x86_64, mingw32
##  ui       RStudio
##  language (EN)
##  collate  English_United States.utf8
##  ctype    English_United States.utf8
##  tz       America/Chicago
##  date     2022-08-05
##  rstudio  2022.02.3+492 Prairie Trillium (desktop)
##  pandoc   2.17.1.1 @ C:/Program Files/RStudio/bin/quarto/bin/ (via rmarkdown)
## 
## ─ Packages ────────────────────────────────────────────────────────────────────────────────────────
##  package       * version date (UTC) lib source
##  abind           1.4-5   2016-07-21 [1] CRAN (R 4.2.0)
##  ape             5.6-2   2022-03-02 [1] CRAN (R 4.2.0)
##  assertthat      0.2.1   2019-03-21 [1] CRAN (R 4.2.0)
##  backports       1.4.1   2021-12-13 [1] CRAN (R 4.2.0)
##  base64enc       0.1-3   2015-07-28 [1] CRAN (R 4.2.0)
##  bezier          1.1.2   2018-12-14 [1] CRAN (R 4.2.0)
##  bookdown        0.27    2022-06-14 [1] CRAN (R 4.2.0)
##  brew            1.0-7   2022-02-04 [1] CRAN (R 4.2.0)
##  broom           1.0.0   2022-07-01 [1] CRAN (R 4.2.0)
##  bslib           0.4.0   2022-07-16 [1] CRAN (R 4.2.1)
##  cachem          1.0.6   2021-08-19 [1] CRAN (R 4.2.0)
##  callr           3.7.1   2022-07-13 [1] CRAN (R 4.2.1)
##  car             3.1-0   2022-06-15 [1] CRAN (R 4.2.0)
##  carData         3.0-5   2022-01-06 [1] CRAN (R 4.2.0)
##  cellranger      1.1.0   2016-07-27 [1] CRAN (R 4.2.0)
##  cli             3.3.0   2022-04-25 [1] CRAN (R 4.2.0)
##  cluster       * 2.1.3   2022-03-28 [2] CRAN (R 4.2.1)
##  colorspace      2.0-3   2022-02-21 [1] CRAN (R 4.2.0)
##  crayon          1.5.1   2022-03-26 [1] CRAN (R 4.2.0)
##  DBI             1.1.3   2022-06-18 [1] CRAN (R 4.2.0)
##  dbplyr          2.2.1   2022-06-27 [1] CRAN (R 4.2.0)
##  devtools        2.4.4   2022-07-20 [1] CRAN (R 4.2.1)
##  digest          0.6.29  2021-12-01 [1] CRAN (R 4.2.0)
##  dplyr         * 1.0.9   2022-04-28 [1] CRAN (R 4.2.0)
##  ellipsis        0.3.2   2021-04-29 [1] CRAN (R 4.2.0)
##  EnvStats      * 2.7.0   2022-03-07 [1] CRAN (R 4.2.0)
##  evaluate        0.15    2022-02-18 [1] CRAN (R 4.2.0)
##  fansi           1.0.3   2022-03-24 [1] CRAN (R 4.2.0)
##  fastmap         1.1.0   2021-01-25 [1] CRAN (R 4.2.0)
##  forcats       * 0.5.1   2021-01-27 [1] CRAN (R 4.2.0)
##  fs              1.5.2   2021-12-08 [1] CRAN (R 4.2.0)
##  gargle          1.2.0   2021-07-02 [1] CRAN (R 4.2.0)
##  generics        0.1.3   2022-07-05 [1] CRAN (R 4.2.0)
##  geomorph      * 4.0.4   2022-07-13 [1] Github (geomorphR/geomorph@dd331e2)
##  ggExtra       * 0.10.0  2022-03-23 [1] CRAN (R 4.2.0)
##  ggfortify     * 0.4.14  2022-01-03 [1] CRAN (R 4.2.0)
##  ggplot2       * 3.3.6   2022-05-03 [1] CRAN (R 4.2.0)
##  ggpubr        * 0.4.0   2020-06-27 [1] CRAN (R 4.2.0)
##  ggsignif        0.6.3   2021-09-09 [1] CRAN (R 4.2.0)
##  git2r           0.30.1  2022-03-16 [1] CRAN (R 4.2.0)
##  glue            1.6.2   2022-02-24 [1] CRAN (R 4.2.0)
##  googledrive     2.0.0   2021-07-08 [1] CRAN (R 4.2.0)
##  googlesheets4   1.0.0   2021-07-21 [1] CRAN (R 4.2.0)
##  gridExtra       2.3     2017-09-09 [1] CRAN (R 4.2.0)
##  gtable          0.3.0   2019-03-25 [1] CRAN (R 4.2.0)
##  haven           2.5.0   2022-04-15 [1] CRAN (R 4.2.0)
##  here          * 1.0.1   2020-12-13 [1] CRAN (R 4.2.0)
##  highr           0.9     2021-04-16 [1] CRAN (R 4.2.0)
##  hms             1.1.1   2021-09-26 [1] CRAN (R 4.2.0)
##  htmltools       0.5.3   2022-07-18 [1] CRAN (R 4.2.1)
##  htmlwidgets     1.5.4   2021-09-08 [1] CRAN (R 4.2.0)
##  httpuv          1.6.5   2022-01-05 [1] CRAN (R 4.2.0)
##  httr            1.4.3   2022-05-04 [1] CRAN (R 4.2.0)
##  jpeg            0.1-9   2021-07-24 [1] CRAN (R 4.2.0)
##  jquerylib       0.1.4   2021-04-26 [1] CRAN (R 4.2.0)
##  jsonlite        1.8.0   2022-02-22 [1] CRAN (R 4.2.0)
##  knitr           1.39    2022-04-26 [1] CRAN (R 4.2.0)
##  later           1.3.0   2021-08-18 [1] CRAN (R 4.2.0)
##  lattice         0.20-45 2021-09-22 [2] CRAN (R 4.2.1)
##  lifecycle       1.0.1   2021-09-24 [1] CRAN (R 4.2.0)
##  lubridate       1.8.0   2021-10-07 [1] CRAN (R 4.2.0)
##  magrittr        2.0.3   2022-03-30 [1] CRAN (R 4.2.0)
##  MASS            7.3-57  2022-04-22 [2] CRAN (R 4.2.1)
##  Matrix        * 1.4-1   2022-03-23 [2] CRAN (R 4.2.1)
##  memoise         2.0.1   2021-11-26 [1] CRAN (R 4.2.0)
##  mime            0.12    2021-09-28 [1] CRAN (R 4.2.0)
##  miniUI          0.1.1.1 2018-05-18 [1] CRAN (R 4.2.0)
##  modelr          0.1.8   2020-05-19 [1] CRAN (R 4.2.0)
##  munsell         0.5.0   2018-06-12 [1] CRAN (R 4.2.0)
##  nlme            3.1-157 2022-03-25 [2] CRAN (R 4.2.1)
##  pillar          1.8.0   2022-07-18 [1] CRAN (R 4.2.1)
##  pkgbuild        1.3.1   2021-12-20 [1] CRAN (R 4.2.0)
##  pkgconfig       2.0.3   2019-09-22 [1] CRAN (R 4.2.0)
##  pkgload         1.3.0   2022-06-27 [1] CRAN (R 4.2.0)
##  png             0.1-7   2013-12-03 [1] CRAN (R 4.2.0)
##  prettyunits     1.1.1   2020-01-24 [1] CRAN (R 4.2.0)
##  processx        3.7.0   2022-07-07 [1] CRAN (R 4.2.1)
##  profvis         0.3.7   2020-11-02 [1] CRAN (R 4.2.1)
##  promises        1.2.0.1 2021-02-11 [1] CRAN (R 4.2.0)
##  ps              1.7.1   2022-06-18 [1] CRAN (R 4.2.0)
##  purrr         * 0.3.4   2020-04-17 [1] CRAN (R 4.2.0)
##  R6              2.5.1   2021-08-19 [1] CRAN (R 4.2.0)
##  Rcpp            1.0.9   2022-07-08 [1] CRAN (R 4.2.1)
##  readr         * 2.1.2   2022-01-30 [1] CRAN (R 4.2.0)
##  readxl          1.4.0   2022-03-28 [1] CRAN (R 4.2.0)
##  remotes         2.4.2   2021-11-30 [1] CRAN (R 4.2.0)
##  reprex          2.0.1   2021-08-05 [1] CRAN (R 4.2.0)
##  rgl           * 0.109.6 2022-07-08 [1] CRAN (R 4.2.1)
##  rjson           0.2.21  2022-01-09 [1] CRAN (R 4.2.0)
##  rlang           1.0.4   2022-07-12 [1] CRAN (R 4.2.1)
##  rmarkdown       2.14    2022-04-25 [1] CRAN (R 4.2.0)
##  Rook            1.1-1   2014-10-20 [1] CRAN (R 4.2.0)
##  rprojroot       2.0.3   2022-04-02 [1] CRAN (R 4.2.0)
##  RRPP          * 1.3.0   2022-06-20 [1] Github (mlcollyer/RRPP@e75eca0)
##  rstatix         0.7.0   2021-02-13 [1] CRAN (R 4.2.0)
##  rstudioapi      0.13    2020-11-12 [1] CRAN (R 4.2.0)
##  rvest           1.0.2   2021-10-16 [1] CRAN (R 4.2.0)
##  sass            0.4.2   2022-07-16 [1] CRAN (R 4.2.1)
##  scales          1.2.0   2022-04-13 [1] CRAN (R 4.2.0)
##  sessioninfo     1.2.2   2021-12-06 [1] CRAN (R 4.2.0)
##  shiny           1.7.2   2022-07-19 [1] CRAN (R 4.2.1)
##  StereoMorph   * 1.6.7   2022-05-25 [1] CRAN (R 4.2.0)
##  stringi         1.7.8   2022-07-11 [1] CRAN (R 4.2.1)
##  stringr       * 1.4.0   2019-02-10 [1] CRAN (R 4.2.0)
##  svgViewR        1.4.1   2020-10-03 [1] CRAN (R 4.2.0)
##  tibble        * 3.1.8   2022-07-22 [1] CRAN (R 4.2.1)
##  tidyr         * 1.2.0   2022-02-01 [1] CRAN (R 4.2.0)
##  tidyselect      1.1.2   2022-02-21 [1] CRAN (R 4.2.0)
##  tidyverse     * 1.3.2   2022-07-18 [1] CRAN (R 4.2.1)
##  tiff            0.1-11  2022-01-31 [1] CRAN (R 4.2.0)
##  tzdb            0.3.0   2022-03-28 [1] CRAN (R 4.2.0)
##  urlchecker      1.0.1   2021-11-30 [1] CRAN (R 4.2.1)
##  usethis         2.1.6   2022-05-25 [1] CRAN (R 4.2.0)
##  utf8            1.2.2   2021-07-24 [1] CRAN (R 4.2.0)
##  vctrs           0.4.1   2022-04-13 [1] CRAN (R 4.2.0)
##  wesanderson   * 0.3.6   2018-04-20 [1] CRAN (R 4.2.0)
##  withr           2.5.0   2022-03-03 [1] CRAN (R 4.2.0)
##  xfun            0.31    2022-05-10 [1] CRAN (R 4.2.0)
##  xml2            1.3.3   2021-11-30 [1] CRAN (R 4.2.0)
##  xtable          1.8-4   2019-04-21 [1] CRAN (R 4.2.0)
##  yaml            2.3.5   2022-02-21 [1] CRAN (R 4.2.0)
## 
##  [1] C:/Users/seldenjrz/AppData/Local/R/win-library/4.2
##  [2] C:/Program Files/R/R-4.2.1/library
## 
## ───────────────────────────────────────────────────────────────────────────────────────────────────</code></pre>
<p>Current Git commit details are:</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="index.html#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="co"># where can I find this commit? </span></span>
<span id="cb6-2"><a href="index.html#cb6-2" aria-hidden="true" tabindex="-1"></a><span class="cf">if</span> (<span class="st">&quot;git2r&quot;</span> <span class="sc">%in%</span> <span class="fu">installed.packages</span>() <span class="sc">&amp;</span> git2r<span class="sc">::</span><span class="fu">in_repository</span>(<span class="at">path =</span> <span class="st">&quot;.&quot;</span>)) git2r<span class="sc">::</span><span class="fu">repository</span>(here<span class="sc">::</span><span class="fu">here</span>())  </span></code></pre></div>
<pre><code>## Local:    master D:/github/jowell.1
## Remote:   master @ origin (https://github.com/seldenlab/jowell.1)
## Head:     [00705bf] 2022-08-04: &lt;edit index&gt;</code></pre>

</div>
</div>
<h3>References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-RN1655" class="csl-entry">
Adams, Dean C., and Michael L. Collyer. 2015. <span>“<span class="nocase">Permutation Tests for Phylogenetic Comparative Analyses of High-Dimensional Shape Data: What you Shuffle Matters</span>.”</span> <em>Evolution</em> 69 (3): 823–29. <a href="https://doi.org/10.1111/evo.12596">https://doi.org/10.1111/evo.12596</a>.
</div>
<div id="ref-RN11530" class="csl-entry">
Adams, Dean C., Michael L. Collyer, Antigoni Kaliontzopoulou, and Emma Sherratt. 2017. <span>“<span class="nocase">Package geomorph: Geometric Morphometric Analyses of Landmark Data. R package version 3.0.5</span>.”</span> <a href="http://geomorphr.github.io/geomorph/">http://geomorphr.github.io/geomorph/</a>.
</div>
<div id="ref-RN1774" class="csl-entry">
Adams, Dean C., and Erik Otarola-Castillo. 2013. <span>“<span class="nocase">geomorph: An R Package for the Collection and Analysis of Geometric Morphometric Shape Data</span>.”</span> <em>Methods in Ecology and Evolution</em> 4 (4): 393–99. <a href="https://doi.org/10.1111/2041-210x.12035">https://doi.org/10.1111/2041-210x.12035</a>.
</div>
<div id="ref-RN1719" class="csl-entry">
Anderson, M. J., and C. J. F. Ter Braak. 2003. <span>“<span class="nocase">Permutation Tests for Multi-Factoral Analysis of Variance</span>.”</span> <em>Journal of Statistical Computation and Simulation</em> 73 (2): 85–113. <a href="https://doi.org/10.1080=0094965021000015558">https://doi.org/10.1080=0094965021000015558</a>.
</div>
<div id="ref-RN9565" class="csl-entry">
Baken, Erica K., Michael L. Collyer, Antigoni Kaliontzopoulou, and Dean C. Adams. 2021. <span>“<span class="nocase">geomorph v4.0 and gmShiny: Enhanced analytics and a new graphical interface for a comprehensive morphometric experience</span>.”</span> Journal Article. <em>Methods in Ecology and Evolution</em>. <a href="https://doi.org/10.1111/2041-210x.13723">https://doi.org/10.1111/2041-210x.13723</a>.
</div>
<div id="ref-RN11775" class="csl-entry">
Collyer, Michael L., and Dean C. Adams. 2018. <span>“<span class="nocase">RRPP: An R Package for Fitting Linear Models to High-Dimensional Data using Residual Randomization</span>.”</span> <em>Methods in Ecology and Evolution</em> 9 (7): 1772–79. https://doi.org/<a href="https://doi.org/10.1111/2041-210X.13029">https://doi.org/10.1111/2041-210X.13029</a>.
</div>
<div id="ref-RN11107" class="csl-entry">
———. 2020. <span>“Phylogenetically Aligned Component Analysis.”</span> Journal Article. <em>Methods in Ecology and Evolution</em> 12 (2): 359–72. <a href="https://doi.org/10.1111/2041-210x.13515">https://doi.org/10.1111/2041-210x.13515</a>.
</div>
<div id="ref-RN1756" class="csl-entry">
Collyer, Michael L., David J. Sekora, and Dean C. Adams. 2015. <span>“<span class="nocase">A Method for Analysis of Phenotypic Change for Phenotypes Described by High-Dimensional Data</span>.”</span> <em>Heredity</em> 115 (4): 357–65. <a href="https://doi.org/10.1038/hdy.2014.75">https://doi.org/10.1038/hdy.2014.75</a>.
</div>
<div id="ref-RN7041" class="csl-entry">
Foote, M. 1993. <span>“Contributions of Individual Taxa to Overall Morphological Disparity.”</span> Journal Article. <em>Paleobiology</em> 19 (4): 403–19.
</div>
<div id="ref-RN20917" class="csl-entry">
Gandrud, Christopher. 2014. <em>Reproducible Research with r and RStudio</em>. The r Series. London: CRC Press.
</div>
<div id="ref-RN1749" class="csl-entry">
Goodall, Colin. 1991. <span>“<span class="nocase">Procrustes Methods in the Statistical Analysis of Shape</span>.”</span> <em>Journal of the Royal Statistical Society. Series B (Methodological)</em> 53 (2): 285–339.
</div>
<div id="ref-RN11564" class="csl-entry">
Gower, J. C. 1975. <span>“<span>Generalized Procrustes Analysis</span>.”</span> <em>Psychometrika</em> 40 (1): 33–51. https://doi.org/<a href="https://doi.org/10.1007/BF02291478">https://doi.org/10.1007/BF02291478</a>.
</div>
<div id="ref-RN20915" class="csl-entry">
Gray, Charles T., and Ben Marwick. 2019. <span>“Truth, Proof, and Reproducibility: There’s No Counter-Attack for the Codeless.”</span> In <em>Statistics and Data Science</em>, 111–29. Communications in Computer and Information Science. <a href="https://doi.org/10.1007/978-981-15-1960-4_8">https://doi.org/10.1007/978-981-15-1960-4_8</a>.
</div>
<div id="ref-RN21009" class="csl-entry">
Ivanovaite, Livija, Kamil Serwatka, Christian Steven Hoggard, Florian Sauer, and Felix Riede. 2020. <span>“All These Fantastic Cultures? Research History and Regionalization in the Late Palaeolithic Tanged Point Cultures of Eastern Europe.”</span> <em>European Journal of Archaeology</em> 23 (2): 162–85. <a href="https://doi.org/10.1017/eaa.2019.59">https://doi.org/10.1017/eaa.2019.59</a>.
</div>
<div id="ref-RN1746" class="csl-entry">
Jolliffe, Ian T. 2002. <em>Principal Component Analysis</em>. New York: Springer.
</div>
<div id="ref-RN11622" class="csl-entry">
Kendall, David G. 1981. <span>“<span class="nocase">The Statistics of Shape</span>.”</span> In <em>Interpreting Multivariate Data</em>, edited by V. Barnett, 75–80. New York: Wiley.
</div>
<div id="ref-RN11623" class="csl-entry">
———. 1984. <span>“<span class="nocase">Shape Manifolds, Procrustean Metrics, and Complex Projective Spaces</span>.”</span> <em>Bulletin of the London Mathematical Society</em> 16 (2): 81–121. <a href="https://doi.org/10.1112/blms/16.2.81">https://doi.org/10.1112/blms/16.2.81</a>.
</div>
<div id="ref-RN1731" class="csl-entry">
Klingenberg, Christian Peter. 2013. <span>“<span class="nocase">Visualizations in Geometric Morphometrics: How to Read and How to Make Graphs Showing Shape Changes</span>.”</span> <em>Hystrix</em> 24 (1): 15–24. https://doi.org/<a href="http://dx.doi.org/10.4404/hystrix-24.1-7691">http://dx.doi.org/10.4404/hystrix-24.1-7691</a>.
</div>
<div id="ref-RN20918" class="csl-entry">
Leek, Jeffrey T., and Roger D. Peng. 2015. <span>“Opinion: Reproducible Research Can Still Be Wrong: Adopting a Prevention Approach.”</span> <em>Proc Natl Acad Sci U S A</em> 112 (6): 1645–46. <a href="https://doi.org/10.1073/pnas.1421412111">https://doi.org/10.1073/pnas.1421412111</a>.
</div>
<div id="ref-RN20804" class="csl-entry">
Marwick, Ben. 2016. <span>“Computational Reproducibility in Archaeological Research: Basic Principles and a Case Study of Their Implementation.”</span> <em>Journal of Archaeological Method and Theory</em> 24 (2): 424–50. <a href="https://doi.org/10.1007/s10816-015-9272-9">https://doi.org/10.1007/s10816-015-9272-9</a>.
</div>
<div id="ref-RN21007" class="csl-entry">
Marwick, Ben, Li-Ying Wang, Ryan Robinson, and Hope Loiselle. 2019. <span>“How to Use Replication Assignments for Teaching Integrity in Empirical Archaeology.”</span> <em>Advances in Archaeological Practice</em> 8 (1): 78–86. <a href="https://doi.org/10.1017/aap.2019.38">https://doi.org/10.1017/aap.2019.38</a>.
</div>
<div id="ref-RN11459" class="csl-entry">
Miller, D. Shane. 2007. <span>“Site Formation Processes in an Upland Paleoindian Site: The 2005 – 2007 Topper Firebreak Excavations.”</span> Thesis.
</div>
<div id="ref-RN11460" class="csl-entry">
Miller, D. Shane, and Ashley M. Smallwood. 2012. <span>“Beyond Stages: Modeling Clovis Biface Production at the Topper Site (38al23), South Carolina.”</span> Book Section. In <em>Lithic Analysis: Problems, Solutions, and Interpretations</em>, edited by Philip J. Carr and Andrew P. Bradbury, 28–41. Tuscaloosa: University of Alabama Press.
</div>
<div id="ref-RN11461" class="csl-entry">
Millsap, Mike, and Don R. Dickson. 1968. <span>“The Millsap Cache.”</span> Journal Article. <em>Bulletin of the Oklahoma Anthropological Society</em> 16: 159–60.
</div>
<div id="ref-RN8973" class="csl-entry">
Olsen, Aaron M., and Mark W. Westneat. 2015. <span>“StereoMorph: An r Package for the Collection of 3d Landmarks and Curves Using a Stereo Camera Set‐up.”</span> Journal Article. <em>Methods in Ecology and Evolution</em> 6 (3): 351–56. <a href="https://doi.org/10.1111/2041-210x.12326">https://doi.org/10.1111/2041-210x.12326</a>.
</div>
<div id="ref-RN21008" class="csl-entry">
Peels, Rik, and Lex Bouter. 2018. <span>“Humanities Need a Replication Drive Too.”</span> <em>Nature</em> 558 (7710): 372. <a href="https://doi.org/10.1038/d41586-018-05454-w">https://doi.org/10.1038/d41586-018-05454-w</a>.
</div>
<div id="ref-RN20916" class="csl-entry">
Peng, Roger D. 2011. <span>“Reproducible Research in Computational Science.”</span> <em>Science</em> 334 (6060): 1226–27. <a href="https://doi.org/10.1126/science.1213847">https://doi.org/10.1126/science.1213847</a>.
</div>
<div id="ref-R" class="csl-entry">
R Core Development Team,. 2022. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="http://www.R-project.org/">http://www.R-project.org/</a>.
</div>
<div id="ref-RN1646" class="csl-entry">
Rohlf, F. James. 1999. <span>“<span class="nocase">Shape Statistics: Procrustes Superimpositions and Tangent Spaces</span>.”</span> <em>Journal of Classification</em> 16 (2): 197–223. <a href="https://doi.org/10.1007/s003579900054">https://doi.org/10.1007/s003579900054</a>.
</div>
<div id="ref-RN478" class="csl-entry">
Rohlf, F. James, and Dennis Slice. 1990. <span>“<span class="nocase">Extensions of the Procrustes Method for the Optimal Superimposition of Landmarks</span>.”</span> <em>Systematic Zoology</em> 39 (1): 40–59. <a href="https://doi.org/10.2307/2992207">https://doi.org/10.2307/2992207</a>.
</div>
<div id="ref-RN9364" class="csl-entry">
Selden Jr, Robert Z., John E. Dockall, C. Britt Bousman, and Timothy K. Perttula. 2021a. <span>“<span class="nocase">Shape as a function of time + raw material + burial context? An exploratory analysis of Perdiz arrow points from the ancestral Caddo area of the American Southeast</span>.”</span> Journal Article. <em>Journal of Archaeological Science: Reports</em> 37: 102916. <a href="https://doi.org/10.1016/j.jasrep.2021.102916">https://doi.org/10.1016/j.jasrep.2021.102916</a>.
</div>
<div id="ref-RN21001" class="csl-entry">
Selden Jr., Robert Z., John E. Dockall, and Morgane Dubied. 2020. <span>“A Quantitative Assessment of Intraspecific Morphological Variation in Gahagan Bifaces from the Southern Caddo Area and Central Texas.”</span> <em>Southeastern Archaeology</em> 39 (2): 125–45. <a href="https://doi.org/10.1080/0734578x.2020.1744416">https://doi.org/10.1080/0734578x.2020.1744416</a>.
</div>
<div id="ref-RN11783" class="csl-entry">
Selden Jr., Robert Z., John E. Dockall, and Harry J. Shafer. 2018. <span>“<span class="nocase">Lithic Morphological Organisation: Gahagan Bifaces from the Southern Caddo Area</span>.”</span> <em>Digital Applications in Archaeology and Cultural Heritage</em> 10: e00080. <a href="https://doi.org/10.1016/j.daach.2018.e00080">https://doi.org/10.1016/j.daach.2018.e00080</a>.
</div>
<div id="ref-RN479" class="csl-entry">
Sherratt, Emma, David J. Gower, Christian Peter Klingenberg, and Mark Wilkinson. 2014. <span>“<span class="nocase">Evolution of Cranial Shape in Caecilians (Amphibia: Gymnophiona)</span>.”</span> <em>Evolutionary Biology</em> 41 (4): 528–45. https://doi.org/<a href="https://doi.org/10.1007/s11692-014-9287-2">https://doi.org/10.1007/s11692-014-9287-2</a>.
</div>
<div id="ref-RN11563" class="csl-entry">
Slice, Dennis E. 2001. <span>“<span class="nocase">Landmark Coordinates Aligned by Procrustes Analysis Do Not Lie in Kendall’s Shape Space</span>.”</span> <em>Systematic Biology</em> 50 (1): 141–49. <a href="https://doi.org/10.1080/10635150119110">https://doi.org/10.1080/10635150119110</a>.
</div>
<div id="ref-RN8310" class="csl-entry">
Smallwood, Ashley M. 2010. <span>“Clovis Biface Technology at the Topper Site, South Carolina: Evidence for Variation and Technological Flexibility.”</span> Journal Article. <em>Journal of Archaeological Science</em> 37 (10): 2413–25. <a href="https://doi.org/10.1016/j.jas.2010.05.002">https://doi.org/10.1016/j.jas.2010.05.002</a>.
</div>
<div id="ref-RN5694" class="csl-entry">
Zelditch, Miriam Leah, Donald L. Swiderski, H. David Sheets, and William L. Fink. 2004. <em>Geometric Morphometrics for Biologists : A Primer</em>. Book. Burlington: Elsevier Science. <a href="http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308">http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308</a>.
</div>
</div>
<div class="footnotes">
<hr />
<ol start="1">
<li id="fn1"><p>Heritage Research Center, Stephen F. Austin State University; Department of Biology, Stephen F. Austin State University; and Cultural Heritage Department, Jean Monnet University, <a href="mailto:zselden@sfasu.edu" class="email">zselden@sfasu.edu</a><a href="#fnref1" class="footnote-back">↩︎</a></p></li>
<li id="fn2"><p>Stantec, Inc.<a href="#fnref2" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="define-size-classes.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "night",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["supp.pdf", "supp.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
